<html><head><title>Revision of the Ictiobinae: Allozymes & Isozymes</title><meta name="generator" content="Frontier 4.2 Mac"></head><body alink="#008000" bgcolor="#ffffff" link="#000fff" vlink="#ff0000"><table width=600 border=0 cols=2 cellpadding=2 cellspacing=0>	<tr valign=top>		<td width=500 align=left>			<center>			<IMG alt="ictiologo picture" border=0 height=105  src="images/ictiologo.jpg" width =365>			</center>			<p>			<center><h3>Allozymes and Isozymes (Don Buth, UCLA)</h3></center><p>	We will use established methods of enzyme electrophoresis to develop two additional independent data sets on intra- and interspecific variation within the genera <i>Carpiodes</i> and <i>Ictiobus</i>.  The catostomid fishes are of tetraploid origin (Uyeno and Smith 1972) and the Ictiobinae still express more than half of their genes in duplicate (Ferris and Whitt 1977).  Patterns of duplicate gene expression and gene silencing have been useful in catostomid systematics (e.g. Ferris and Whitt 1978, Buth 1979). To date, such gene expression patterns have been reported to be species specific.  No intraspecific variation in gene number has been noted except for the related catostomid <i>Cycleptus elongatus</i> (Buth and Mayden 1998) and these patterns have been interpreted as those of three allopatric species whose biodiversity is currently hidden under a single taxon. We now have the methods to more than double the number of enzyme systems (e.g. Buth et al. 1993, 1996) that were used by Ferris and Whitt (1977) to compare catostomid fishes.  We propose to generate such a robust gene expression data set for the Ictiobinae.  Outgroup comparisons can be made with <i>Cycleptus elongatus</i> (Mayden and Buth, in prep.), and <i>Myxocyprinus asiaticus</i> (Tsoi et al. 1989, Buth unpubl. data).  Gene expression data are ideal for maximum parsimony analyses (Ferris and Whitt 1978).</p><p>	Allozymic data have also been useful in catostomid systematics, however most applications have been made among the Catostominae (e.g. 20+ articles by the PI (Buth) including his dissertation).  In several cases, allozymic data revealed the existence of multiple species under a single name (e.g. Buth 1978, Ferris et al. 1982, Buth and Timmons 1992, Buth and Mayden 1998).  The procedures we intend to employ for the generation and comparison of an allozymic data set for the Ictiobinae are those of Murphy et al. (1996).   Gene products of about 75 loci can be scored from separate extracts of brain, liver, heart, and skeletal muscle.  We intend to score 10 individuals of each currently recognized species from the regions listed in Table 5 (ca. 600 specimens).  This is basically the same sampling strategy used in a previous study of darters (<i>Catonotus</i>) that revealed five new species in the group (Page et al. 1992).  Allozymic and isozymic information will be taken from the same specimens sampled for molecular sequencing so that direct comparisons can be made.   Phylogenetic analyses of the allozymic data will first assess homogeneous units within the data set (e.g. as for <i>Pungitius</i>, Haglund et al. 1992), followed by variable coding procedures (e.g. Buth 1984, Mabee and Humphries 1993) and maximum parsimony analyses of each kind of coding.</p><center><h3>Preliminary allozyme/isozyme study</h3></center><p>	We examined gene expression and allozymic characteristics of a limited number of <i>Carpiodes carpio, C. cyprinus, C. velifer, Ictiobus bubalus, I. cyprinellus, I. niger, Cycleptus elongatus and Myxocyprinus asiaticus.</i>  Sixty gene products were resolved from two tissue sources (liver and skeletal muscle).  In some cases, specimens of a species from different river systems were compared.</p><p>	We found two cases in which the number of genes controlling a given enzyme system differed from those reported by Ferris and Whitt (1977) for <i>Carpiodes</i>.  These differences could be due to scoring errors (to be resolved via another replication), or it could be that we are examining different (unrecognized) species of <i>Carpiodes</i> because our samples came from locations different than the sources used by Ferris and Whitt (1977).  Additional geographic sampling will address this problem.</p><p>	Ingroup allozyme expression ranged from monoallelic loci, to intergeneric differences, to species specific differences, and to highly variable loci that will be useful for population-level comparisons. The two outgroup species did not share a lot of alleles with the ingroup species, however several were shared so that rooting a cladogram that is based on these data is still possible. Species of <i>Ictiobus</i> showed fewer interspecific allelic differences than did species of <i>Carpiodes</i>. A striking result of the preliminary study is our observation that <i>Carpiodes cyprinus</i> from the Wisconsin River (WI) and the Savannah River (GA/SC) are completely different at several loci and even have gene duplication differences between them.  A wider geographic comparison will allow us to determine whether we are dealing with separate species.   The two specimens of <i>"C. forbesi?</i>" from the Wisconsin River we examined were genetically identical to <i>C. cyprinus</i> from that drainage.  However, the identifications of the former were in question and additional comparisons are necessary before the question of recognition or synonymy is answered. </p>      <P></P>			<p align=right>			<A href="morphometrics.html">Prev</A> | <A href="file://G:\ictiobin\DNA.html" >Next</A></p>		</td>		<td width=100 align=right bgcolor="#99cc99">			<center>			<IMG alt="tree picture" border=0 height=77  src="images/tree.gif" width =44>			</center>			<font face=arial,geneva>			<p>			<A href="default.html">Summary</A>			</p>			<p>			<A href="intro.html">Introduction</A>			</p>			<p>			<A href="carpiodes.html">Carpiodes</A>			</p>			<p>			<A href="ictiobus.html">Ictiobus</A>			</p>			<p>			<A href="distribution.html">Distribution</A>			</p>			<p>			<A href="field.html">Field Work</A>			<p>			<A href="extmorph.html">External Morphology</A>			</p>			<p>			<A href="typeloc.html">Types &amp; Their Location</A>			</p>			<p>			<A href="morphometrics.html">Morphometrics</A>			</p>			<p>			<A href="allozymes.html">Allozymes &amp; Isozymes</A>			</p>			<p>			<A href="file://G:\ictiobin\DNA.html" >Molecular Sequencing</A>			</p>			<p>			<A href="litcited.html">Literature Cited</A>			</p>			<p>			<A href="pubpres.html">Publications &amp; Presentations</A>			</p>			<p>			<a href="https://www.tubri.org/museum/museum/reshank.html">Hank Bart's Research</a>			</p>			<p>			<a href="https://www.tubri.org/museum/museum/museum.html">Tulane Museum of Natural History</a>			</p>			</font>		</td>	</tr></table><p><IMG src="counter.count$tulane/ictiobin"></p></body></html><!--#fatPage#version 1#docs http://www.scripting.com/fatPages/faq.html#adrObject user.websites.ictiobin.allozymes#netAddress 129.81.102.17#objectCreated 1/15/99; 12:40:18 PM#objectModified 