<html><head><title>Revision of the Ictiobinae: mtDNA Sequencing</title><meta name="generator" content="Frontier 4.2 Mac"></head><body alink="#008000" bgcolor="#FFFFFF" link="#000FFF" vlink="#FF0000"><table width=600 border=0 cols=2 cellpadding=2 cellspacing=0>	<tr valign=top>		<td width=500 align=left>			<center>			<img src="images/ictiologo.jpg" height=105 width=365 border=0  alt="ictiologo picture" border=0>			</center>			<p>			<center><h3>Molecular sequencing</h3></center>   <p>	We will use methods of molecular systematics to establish an additional data set on intra- and interspecific genetic variation within the genera Carpiodes and Ictiobus. For the past year, Bart and his graduate student, Kyle R. Piller, have been learning molecular sequencing techniques in the laboratory of Dr. David Hurley.</p><p>	The cytochrome <i>b</i> gene has proven to be useful for inferring phylogenetic relationships in a variety of vertebrate groups, including actinopterygian fishes (Song 1994, Schmidt and Gold 1995, Lydeard et al. 1995, Dowling and Naylor 1997, Lydeard and Roe 1997, Wiley and Hagen 1997, Schmidt et al. 1998).   The gene has both conservative and variable regions, useful for investigating relationships of both distantly and closely related taxa.  The entire cytochrome <i>b</i> gene will be sequenced and the data used to infer phylogenetic relationships within <i>Carpiodes</i> and <i>Ictiobus</i>. </p><p>	Procedures have been developed for extracting mitochondrial DNA from liver and muscle tissue using a commercial DNA ealign=rightxtraction kit (Bio101, Inc.).  The tissue is first homogenized to produce dispersed single cells.  The homogenate is then treated with RNAse before lysis with detergent.  Proteinase is then added, followed by selective precipitation of genomic DNA in 100% ethanol.  The precipitated DNA is then dried, re-suspended in water, and quantified to determine purity and amount.</p><p>	Primers have been selected that amplify the cytochrome <i>b</i> region (1,140 bp) of mitochondrial DNA (Table 1 and Figure: <a href="figures/primer.html">Cytochrome b Primers</a>).  The primers were designed by Tom Near of the Illinois Natural History Survey, who used them amplify the cytochrome <i>b</i> region in other Actinopterygian fishes, including percids and cyprinids.  One of the primers (THR) was subsequently modified to better amplify the cytochrome <i>b</i> region of catostomids.  In an effort to obtain additional sequence data, we used OLIGO Primer Analysis Software to create an additional DNA sequencing primer (904R, Figure: <a href="figures/primer.html">Cytochrome b Primers</a>) which was also used in our preliminary sequencing study.</p><table border="2" cellpadding="0" cellspacing="0" width="495" cols=2><tr valign=top>	<td align=left valign=top width="495" colspan=2>Table 1. Primers for PCR ampilification and DNA sequencing of cytochrome <i>b</i>.</td></tr><tr valign=top>	<td align=left valign=top width="495" colspan=2><b>Primers for PCR amplification</b></td></tr><tr valign=top>	<td align=left valign=top width="395">GLU  5'- TCCGACATTCGGTTTACAAG-3'</td>	<td align=left valign=top width="100">(T. Near, INHS, unpubl.)</td></tr><tr valign=top>	<td align=left valign=top width="395">THR  5'- GACTTGAAGAACCACCGTTG-3'<br> &nbsp;&nbsp;&nbsp;modified</td>	<td align=left valign=top width="100">(T. Near, INHS, unpubl.)</td></tr><tr valign=top>	<td align=left valign=top width="495" colspan=2><b>Primers for DNA sequencing</b></td></tr><tr valign=top>	<td align=left valign=top width="395">904R  5'-GATCGTAGGATGGCGTAGG-3'</td>	<td align=left valign=top width="100">(This study)</td></tr></table><p>Double stranded product is produced using 100 ng of total DNA, 40 pmol of each primer, 0.5mM MgCl<sub>2</sub>, 5&#181;l 10x reaction buffer (Perkin Elmer), 1 &#181;l dNTP's, 1 ml <i>Taq</i> polymerase, and water to bring the reactions to a final volume of 50&#181;l.  Reactions are cycled according to the following temperature profile: 94&#176;C/1 min., 68&#176;C/1 min., and 72&#176;C/1 min. (ca. 30 cycles).  Amplification efficiency is monitored by electrophoresing 10 &#181;l of PCR product through a 0.7% agarose gel.  The band containing the amplified DNA is cut from the agarose gel and purified with the Gene Clean II Kit (Bio101, Inc.).</p><p> PCR products are inserted into plasmid vectors following the protocol of the TA Cloning System (Invitrogen Corp.).  Ligated DNA is transformed into competent cells and screened on 2xYT plates containing 50 &#181;l/mg each of ampicillin and kanamycin.  Transformed colonies are grown in TB containing 200 &#181;l/mg ampicillin.  Plasmid DNA is purified with the Perfect Prep Kit (5'-3', Inc.).  Cloned PCR products are used for <i>Taq</i> DyeDeoxy Terminator cycle-sequencing reactions.  Sequencing reactions are set up using 50 ng template DNA, 20 pmol sequencing primer, and 8&#181;l fluorescent dye-labeled dideoxy terminators with Amplitaq DNA Polymerase (Perkin-Elmer), and cycled according to the manufacturers recommendations.</p><p>Plasmid DNA samples are precipitated in ethanol.  Three microliters of formamide are then added to each sample. Fragments are electrophoretically separated on 6% polyacrylamide gels (8 M urea).  Samples are electrophoresed on the Applied Systems 373A DNA Sequencing System.  Approximately 370-420 bp are obtained per sample.</p><p>Sequence data are evaluated for ambiguities and "clear" sequence length using  Factura 1.2.Or6 (Applied Biosystems, Inc.).  Next, multiple simultaneous alignments of sequences are created using Gene Works 2.45 (Oxford Biomolecular).  All Ictiobine sequences are aligned to <i>Cyprinus carpio</i> (GenBank X61010).  The aligned sequence data are subjected to a maximum parsimony analysis using a variety of search algorithms and weighting schemes using PAUP* (Swofford 1998).  In our preliminary study of the relationships of <i>Carpiodes</i> described below, phylogenetic trees are rooted by the outgroup method (Maddison et al. 1984) using sequences from <i>Ictiobus bubalus</i> and <i>Myxocyprinus asiaticus</i>.</p><center><h3>Preliminary sequencing work</h3></center><p>	The two primers obtained from Tom Near of the Illinois Natural History Survey were used to amplify the mitochondrial cytochrome <i>b</i> region for each of the taxa listed below.  A third sequencing primer was used to obtain additional sequence data from 3' region of cytochrome <i>b</i> (Table 1, Figure: <a href="figures/primer.html">Cytochrome b Primers</a>).</p><table border="0" cellpadding="0" cellspacing="0" width="495"><tr valign=top>	<td align=left valign=top width="175"><i>Myxocyprinus asiaticus</i> </td>	<td align=left valign=top width="320">(Genbank AF036176)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Ictiobus bubalus</i></td>		<td align=left valign=top width="320">(Upper Miss. R. Dr., Wisconsin R., WI)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Upper Miss. R. Dr., Rock R., IL)  </td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Illinois R. Dr. Illinois R., IL)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Lake Pontchartrain Dr., Amite R., LA)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Ohio R. Dr., Kentucky R., KY)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Upper Miss. R., Dr., Wisconsin R., WI)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Upper Miss. R., Dr., Wisconsin R., WI)*<br>&nbsp;&nbsp;&nbsp;"Elongate form"</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes cyprinus</i></td>	<td align=left valign=top width="320">(Chesapeake Bay Dr., James River, VA)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes velifer</i></td>	<td align=left valign=top width="320">(Upper Miss. R. Dr., Wisconsin R., WI)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes velifer</i></td>	<td align=left valign=top width="320">(Lake Pontchartrain Dr., Amite R., LA)</td></tr><tr valign=top>	<td align=left valign=top width="175"><i>Carpiodes carpio</i></td>	<td align=left valign=top width="320">(Ohio R .Dr., Kentucky R., KY)</td></tr></table><p>	A 680 bp fragment of the 3' region of the cytochrome <i>b</i> gene was obtained for each of the above taxa (see <a href="figures/electropherogram.html">Electropherogram</a>). Uncorrected sequence divergence among the ingroup taxa ranged from 1.49 to 9.49% (Mean=4.35%, <a href="tables/table3.html">Table 3</a>).  Roughly 70% of the changes were at the third codon position.  Transitions were recorded three times more often than transversions. </p><p>	Preliminary phylogenetic analysis (PAUP*) utilizing all of the above taxa resulted in a single most parsimonious tree that failed to support the monophyly of <i>C. cyprinus</i> and a close sister relationship of the short-headed, nipple-lip forms, <i>C. carpio</i> and <i>C. velifer</i>, as implied by a recent phylogenetic hypothesis based largely on morphology (Smith 1992).  When the data are weighted according to the observed transition/ transversion ratio (3:1), ten equally parsimonious trees were obtained.  The 50% majority-rule consensus tree (Figure A: <a href="figures/parstrees.html">Weighted Consensus Tree</a>) was also incongruent with the morphology-based phylogeny.</p><p> 	The 3' region of cytochrome <i>b</i> evolves much more rapidly (is more variable) than the 5' portion of the gene (Meyer 1994, Lydeard and Roe 1997).  Data from this segment of cytochrome <i>b</i> appears to be inappropriate for resolving relationships at the level of species complexes within genus <i>Carpiodes</i>.   Our finding of saturation at the third codon position for this region (the position where most of the changes occurred) suggests that some of the similarities among more distantly related taxa likely represent homoplasy.  We are confident that inclusion of sequence data from the more conserved 5' region of cytochrome <i>b</i> will better resolve interrelationships among the species complexes.</p><p>	When the ingroup is limited to only the <i>C. cyprinus</i>-like forms (unweighted parsimony analysis using <i>M. asiaticus</i> and <i>I. bubalus</i> as outgroups), a single tree is obtained  (221 steps, consistency index of 0.68, Figure B: <a href="figures/parstrees.html">Unweighted Phylogeny</a>).  In this tree, all of the elongate-bodied <i>C. cyprinus</i> of the upper Midwest form a monophyletic group sister to the <i>C. cyprinus</i> form from the James River.  The elongated-bodied/James River group is sister to the <i>C. cyprinus</i> form in the Amite River (Lake Pontchartrain Basin), and this group is sister in turn to a deep bodied, long-quilled <i>C. cyprinus</i> form from the Wisconsin River (upper Mississippi River).</p><p>The name <i>C. forbesi</i> was applied by Hubbs (1930) to an elongate-bodied, short quilled, <i>C. cyprinus</i>-like form in the Illinois River, Illinois.  Trautman (1956) described <i>C. cyprinus hinei</i> as an elongate-bodied, short-quilled form inhabiting the Ohio River system in Ohio.  The form inhabiting Lake Erie and surrounding tributaries was regarded as synonymous with a deep-bodied, long-quilled form that ranged from the upper Mississippi River, across the Great Lakes, to the St. Lawrence River and upper Atlantic Slope (Trautman 1956).  Our analysis of partial cytochrome <i>b</i> sequence data for <i>C. cyprinus</i>-like forms, supports the recognition of a distinctive, elongate form in the upper Midwest, which is most closely related to the <i>C. cyprinus</i> form found on the middle Altantic Coast (Bart et al., in preparation).  Moreover, the partial cytochrome <i>b</i> sequence data strongly suggests that a second, more deep-bodied <i>C. cyprinus</i>-like form occurs syntopically with the elongate form in the Wisconsin River. Sequence divergence between these forms is 3.57% (Table: <a href="tables/sequence.html">Pairwise Sequence Divergence</a>).  Perhaps, this second form is the more northerly distributed, deep-bodied form Trautman (1956) referred to as nominal <i>C. cyprinus</i>.</p><center><h3>Proposed sequencing work</h3></center><p>	We will obtain complete cytochrome <i>b</i> sequence data for three specimens drawn from each of several areas of the distribution of each of the currently recognized species (Table: <a href="tables/sampling.html">Tissue Sampling</a>), areas that our morphological studies of ictiobines and evidence from other groups of fishes suggest might represent zones of differentiation (Wiley and Mayden 1985, Mayden 1987).   Sampling areas along the Gulf and Atlantic coasts are broad zones, encompassing a number of rivers that lack freshwater connections today.  In order to initially screen for genetic divergence within these zones, we will obtain sequence data for a single individual from three different river systems within each zone.  Support is requested for contingency sequencing of the cytochrome <i>b</i> gene for 90 additional specimens to allow us further investigate cases of high, within-system or region sequence variability.  Contingency sequencing will be conducted during Year 3 of the project, following review of the sequencing results for the initial 180 specimens.  In the contingency sequencing, we will obtain cytochrome <i>b</i> sequence data for additional specimens from the same populations sampled in the initial screening, or specimens from different populations, in order to clear up confusion or fill in gaps in our understanding of diversity and interrelationships within the species complexes.</p><p>	In order to obtain complete cytochrome <i>b</i> sequence data, we will design five new sequencing primers: two more for sequencing in the 3' > 5' direction, and three for sequencing in the 5'> 3' direction on the complement strand (8 primers total, including the three presently available). </p>			</p>			<p align=right>			<a href="allozymes.html">Prev</a> | <a href="litcited.html">Next</a>		</td>		<td width=100 align=right bgcolor="#99CC99">			<center>			<img src="images/tree.gif" height=77 width=44 border=0  alt="tree picture" border=0>			</center>			<font face=arial,geneva>			<p>			<a href="default.html">Summary</a>			</p>			<p>			<a href="intro.html">Introduction</a>			</p>			<p>			<a href="carpiodes.html">Carpiodes</a>			</p>			<p>			<a href="ictiobus.html">Ictiobus</a>			</p>			<p>			<a href="distribution.html">Distribution</a>			</p>			<p>			<a href="field.html">Field Work</a>			<p>			<a href="extmorph.html">External Morphology</a>			</p>			<p>			<a href="typeloc.html">Types & Their Location</a>			</p>			<p>			<a href="morphometrics.html">Morphometrics</a>			</p>			<p>			<a href="allozymes.html">Allozymes & Isozymes</a>			</p>			<p>			<a href="mtDNA.html">mtDNA Sequencing</a>			</p>			<p>			<a href="litcited.html">Literature Cited</a>			</p>			<p>			<a href="pubpres.html">Publications & Presentations</a>			</p>			<p>			<a href="https://www.tubri.org/museum/museum/reshank.html">Hank Bart's Research</a>			</p>			<p>			<a href="https://www.tubri.org/museum/museum/museum.html">Tulane Museum of Natural History</a>			</p>			</font>		</td>	</tr></table><p><img src="counter.count$tulane/ictiobin"></p></body></html><!--#fatPage#version 1#docs http://www.scripting.com/fatPages/faq.html#adrObject user.websites.ictiobin.mtDNA#netAddress 129.81.102.17#objectCreated 1/9/98; 2:55:35 PM#objectModified 1/22/99; 3:37:03 PM#lastBuildTime 2/25/99; 4:29:46 PM-->