<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 12">
<meta name=Originator content="Microsoft Word 12">
<link rel=File-List href="Abstracts_files/filelist.xml">
<title>Special Symposium: Biology of Cypriniformes</title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Richard Mayden</o:Author>
  <o:LastAuthor>Hank Bart</o:LastAuthor>
  <o:Revision>2</o:Revision>
  <o:TotalTime>424</o:TotalTime>
  <o:Created>2009-10-06T18:06:00Z</o:Created>
  <o:LastSaved>2009-10-06T18:06:00Z</o:LastSaved>
  <o:Pages>9</o:Pages>
  <o:Words>3502</o:Words>
  <o:Characters>21644</o:Characters>
  <o:Company>Saint Louis University</o:Company>
  <o:Lines>180</o:Lines>
  <o:Paragraphs>50</o:Paragraphs>
  <o:CharactersWithSpaces>25096</o:CharactersWithSpaces>
  <o:Version>12.00</o:Version>
 </o:DocumentProperties>
</xml><![endif]-->
<link rel=themeData href="Abstracts_files/themedata.thmx">
<link rel=colorSchemeMapping href="Abstracts_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves/>
  <w:TrackFormatting/>
  <w:DoNotHyphenateCaps/>
  <w:PunctuationKerning/>
  <w:DisplayHorizontalDrawingGridEvery>0</w:DisplayHorizontalDrawingGridEvery>
  <w:DisplayVerticalDrawingGridEvery>0</w:DisplayVerticalDrawingGridEvery>
  <w:ValidateAgainstSchemas>false</w:ValidateAgainstSchemas>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotUnderlineInvalidXML/>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SelectEntireFieldWithStartOrEnd/>
   <w:UseWord2002TableStyleRules/>
   <w:DontUseIndentAsNumberingTabStop/>
   <w:FELineBreak11/>
   <w:WW11IndentRules/>
   <w:DontAutofitConstrainedTables/>
   <w:AutofitLikeWW11/>
   <w:UnderlineTabInNumList/>
   <w:HangulWidthLikeWW11/>
   <w:UseNormalStyleForList/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="--"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="267">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="header"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="footer"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false"
   Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Hyperlink"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Balloon Text"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" SemiHidden="false" UnhideWhenUsed="false"
   QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;
	mso-font-alt:"Arial Unicode MS";
	mso-font-charset:128;
	mso-generic-font-family:modern;
	mso-font-pitch:fixed;
	mso-font-signature:-1610612033 1757936891 16 0 131231 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-1610611985 1107304683 0 0 159 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:1627400839 -2147483648 8 0 66047 0;}
@font-face
	{font-family:"Lucida Grande";
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-format:other;
	mso-font-pitch:variable;
	mso-font-signature:3 0 0 0 1 0;}
@font-face
	{font-family:"\@MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;
	mso-font-charset:128;
	mso-generic-font-family:modern;
	mso-font-pitch:fixed;
	mso-font-signature:-1610612033 1757936891 16 0 131231 0;}
@font-face
	{font-family:TimesNewRomanPSMT;
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:auto;
	mso-font-signature:3 0 0 0 1 0;}
@font-face
	{font-family:Times-Roman;
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:auto;
	mso-font-signature:3 0 0 0 1 0;}
@font-face
	{font-family:TimesNewRomanPS-BoldMT;
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:77;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:auto;
	mso-font-signature:3 0 0 0 1 0;}
@font-face
	{font-family:TimesNewRomanPS-ItalicMT;
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:77;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:auto;
	mso-font-signature:3 0 0 0 1 0;}
@font-face
	{font-family:TimesNRMT;
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:auto;
	mso-font-signature:3 0 0 0 1 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-link:"Header Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 3.0in right 6.0in;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-link:"Footer Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 3.0in right 6.0in;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	mso-style-unhide:no;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:9.0pt;
	font-family:"Lucida Grande";
	mso-fareast-font-family:"Times New Roman";
	mso-bidi-font-family:"Lucida Grande";}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-ascii-font-family:Tahoma;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Header;
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-size:11.0pt;
	mso-ansi-font-size:11.0pt;
	mso-bidi-font-size:11.0pt;}
.MsoPapDefault
	{mso-style-type:export-only;
	margin-bottom:10.0pt;
	line-height:115%;}
@page Section1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
 /* List Definitions */
 @list l0
	{mso-list-id:641809452;
	mso-list-type:hybrid;
	mso-list-template-ids:1299883798 67698703 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l0:level1
	{mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level2
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level3
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:1.5in;
	mso-level-number-position:right;
	text-indent:-9.0pt;}
@list l0:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level5
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level6
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:3.0in;
	mso-level-number-position:right;
	text-indent:-9.0pt;}
@list l0:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level8
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level9
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:4.5in;
	mso-level-number-position:right;
	text-indent:-9.0pt;}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-qformat:yes;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin-top:0in;
	mso-para-margin-right:0in;
	mso-para-margin-bottom:10.0pt;
	mso-para-margin-left:0in;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Times New Roman","serif";}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1026"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link=blue vlink=purple style='tab-interval:.5in'>

<div class=Section1>

<p class=MsoNormal align=center style='text-align:center'><b><i><span
style='font-size:16.0pt'>Special Symposium: International Workshop on Enhancing
Freshwater Fish Biodiversity Research in Africa<o:p></o:p></span></i></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>17-21 November 2008<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>Rhodes University and South African Institute for
Aquatic Biodiversity<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>Grahamstown, South Africa<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>Abstracts<o:p></o:p></span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Michael H Doosey*<sup>1</sup>,
Henry L Bart, Jr<sup>1</sup> and Kenji Saitoh<sup>2<o:p></o:p></sup></span></p>

<p class=MsoNormal><sup>1 </sup>Department of Ecology and Evolutionary Biology,
Tulane University, New Orleans, LA70118, USA and Tulane University Museum of
Natural History, Belle Chasse, LA 70037, USA; <sup>2 </sup>National Research
Institute of Fisheries Science, Aquatic Genomics Research Center, Yokohama
236-8648, JAPAN. E-mail: ksaitoh@affrc.go.jp</p>

<p class=MsoNormal><b><i><o:p>&nbsp;</o:p></i></b></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Systematics of
Catostomidae (Cypriniformes) based on Mitochondrial ND4/ND5 Gene Sequences</span></i>
</b>(poster presentation)<b><o:p></o:p></b></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Abstract: </b>Family Catostomidae (Cypriniformes) is a
diverse group of benthic fresh water fishes that are distributed across North
America and Asia.<span style='mso-spacerun:yes'>  </span>Catostomid
relationships have been inferred from diverse character sets (e.g. molecular,
biochemical, morphology, behavior) by previous researchers.<span
style='mso-spacerun:yes'>  </span>In this study the phylogenetic relationships
of Catostomidae is examined using mitochondrial ND4 and ND5 protein coding
genes.<span style='mso-spacerun:yes'>  </span>Catostomidae and its four
subfamilies are found to be monophyletic; however, the relationships of the
subfamilies are equivocal.<span style='mso-spacerun:yes'>  </span>Relationships
of many genera and species groups are similar to previous hypotheses.<span
style='mso-spacerun:yes'>  </span>ND4 and ND5 genes have better phylogenetic
utility for determining catostomid relationships than cytochrome <i>b</i>.</p>

<p class=MsoNormal><span style='mso-spacerun:yes'> </span><b><o:p></o:p></b></p>

<p class=MsoNormal><b>Keywords: </b>Phylogenetics, mtDNA</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Alice Ekwu <o:p></o:p></span></p>

<p class=MsoHeader style='tab-stops:.5in'>Department of Fisheries and
Aquaculture, University of Uyo, Nigeria</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Observations of </span></i></b><b><span
style='font-size:14.0pt'>Heterotis niloticus<i> in Cross River basin: an
introduced species</i> </span></b>(poster presentation)</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal style='text-align:justify;tab-stops:80.25pt'>Abstract: <i>Heterotis</i>
<i>niloticus</i> (Cuvier), a purely freshwater species of the Ostariophysian
group, is found naturally occurring in Northern Nigerian waters in swamps,
creeks, rivers, lakes and reservoirs. The species was accidentally introduced
into to cross River Basin, close to the Altantic coastal zone in South Eastern
Nigeria, in 1992. Since then, the species has exhibited tremendous success in
the area, dominating the catches of the inland waters, contributing about 50%
of the total fresh fish landings in some places, when in season. Highest body
weight recorded was 8kg and length of 1.2 meters (Highest length and weight
reported in Northern Nigeria waters was 1 meter and 6kg, respectively). The
success of this species in the Cross River basin is attributed to the high
primary productivity of the area, since it is a planktivorous species. A total
of 63 species of Algae with total cell density of 1.17 x 10<sup>3</sup> cells/L
(Dry season) have been observed in the Cross River. Also, highly reduced
conspecific and heterospcific competition in the face of high abundance of
resources may contribute immensely to the success of the species in the basin.
Highest abundance of <i>H. niloticus</i> was observed in the months of
September to November (end of rainy season into the dry season), each annual
cycle. In their indigenous habitat of Northern Nigeria, however, highest
abundance is from June to August each year (rainy season), with rising river
level. Further research on the possible displacement of other species by <i>H</i>.
<i>niloticus</i> and alteration of the gene pool in the Cross River basin is
desirable.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Key words:</b> Abundance, Basin, Cross River, <i>Heterotis
niloticus</i></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><sup><o:p>&nbsp;</o:p></sup></p>

<p class=MsoNormal><span style='font-size:14.0pt'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Thomas Vigliotta<sup>1</sup>,
John Friel<sup>1</sup>*, Alfred Thomson<sup>2</sup>, &amp; Rashid Tamatamah<sup>3<o:p></o:p></sup></span></p>

<p class=MsoNormal><sup>1</sup>Cornell University Museum of Vertebrates,
Ithaca, New York, 14850 USA; <sup>2</sup>Alfred Thomson, Florida Museum of
Natural History, University of Florida, Gainesville, Florida, 32611 USA; <sup>3</sup>Rashid
Tamatamah, Department of Fisheries Science and Aquaculture, University of Dar
es Salaam, Dar es Salaam, United Republic of Tanzania</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Ichthyofaunal Survey of
the Rivers of the Eastern Arc Mountains in Tanzania</span></i></b><span
style='font-size:14.0pt'> </span>(poster presentation)<span style='font-size:
14.0pt'><o:p></o:p></span></p>

<p class=MsoNormal><sup><o:p>&nbsp;</o:p></sup></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b>Between
8 September and 4 October 2007, we conducted a preliminary survey of fish
species in several rivers draining the Eastern Arc Mountains of coastal
Tanzania, including the Umba, Sigi, Pangani, Ruvu, Wami and Ruaha/Rufiji River
basins. The Eastern Arc Mountains are renowned for a high level of endemicity
among plants and terrestrial animals and are considered one of the world's top
conservation &quot;hot spots.&quot; Despite early exploration of this area
during both British and German colonial periods, the ichthyofauna of this
important region is still poorly known. A total of 29 field sites were sampled,
with an emphasis on fast flowing streams arising in the Usambara, Uluguru and
Udzungwa Mountains, individual ranges within the Eastern Arc Mountains. We
employed a variety of methods to collect fishes including seines, cast nets,
dip nets and an electroshocker. Approximately 75 species of fishes, from 17
families, were collected. Rheophilic catfishes like <i>Chiloglanis</i>
(Mochokidae) and <i>Amphilius</i> (Amphiliidae), as well as species of <i>Labeo</i>,
<i>Garra</i> and <i>Barbus</i> (Cyprinidae) are particularly well represented
in these collections. Preliminary assessments of the material indicate the
presence of several new species, including but not limited to the families Amphiliidae,
Mochokidae, Cichlidae, and Mormyridae. The limited sampling of this vast area
and its conservation significance strongly suggest that further collecting
efforts are necessary to accurately and thoroughly document the fishes of this
region. Specimens and associated tissue samples are deposited at the Cornell
University Museum of Vertebrates and Florida Museum of Natural History.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: Tanzania, Amphiliidae, Mochokidae,
Cichlidae, Mormyridae, </p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Philippe Lalèyè<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-GB
style='mso-ansi-language:EN-GB'>Laboratory of Hydrobiology and Aquaculture/University
of Abomey-Calavi, </span>Cotonou Bénin</p>

<p class=MsoNormal style='text-align:justify'><o:p>&nbsp;</o:p></p>

<p class=MsoNormal style='text-align:justify'><b><i><span lang=EN-GB
style='font-size:14.0pt;mso-ansi-language:EN-GB'>Diversity and Status of Fish
Biodiversity in the Oueme river basin, Bénin</span></i></b><span lang=EN-GB
style='mso-ansi-language:EN-GB'> (poster presentation)<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><b><i><span lang=EN-GB
style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></i></b></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b><span
lang=EN-GB style='mso-ansi-language:EN-GB'>Study of the fish fauna of Oueme
river basin stated since 1999 permitted to inventory 124 species of fish
distributed in 87 genera belonging to 51 families. The mormyrids are the most
numerous with 12 species (10 % of the ichthyofauna). Following by the cichlids,
characids, cyprinids and gobiids with respectively 10, 8, 7 and 6 species. The
clariids, cyprinodontids and eleotrids are represented each with 5 species.
Thirty species belonging to 20 families are typically marines or estuarines.
Twenty five species (belonging to 17 families) are added to the fish list known
up to now for the river. Fish richness is unequally distributed along the
river. The lower course of the basin represented by Agonlin Lowe and Toue is by
far the richest (Agonlin Lowe, 71 species, = 59.2% of the total ichthyofauna;
Toue, 67 species, = 55.8% ichtyofauna). This zone concerns a vast floodplain
whose ecological characteristics favour this important colonization by fish.
The stations upstream and on the central course of the river, represented by
Kpassa and Atchakpa have the lowest number of species (47 and 46 species respectively).
The factors that threaten the fish diversity of the Oueme river basin include
mainly degradation of fish habitats due to some human activities or to climate
perturbations and over exploitation.</span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: <span lang=EN-GB style='mso-ansi-language:
EN-GB'>Ichthyofauna, Benin, Ouémé basin, Biodiversity</span></p>

<p class=MsoNormal style='text-align:justify'><sup><o:p>&nbsp;</o:p></sup></p>

<p class=MsoNormal style='text-align:justify'><sup><o:p>&nbsp;</o:p></sup></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:14.0pt;
font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Paula
Mabee*, Ericka Grey, Jeff Engeman<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'>Dept. of Biology, University of South Dakota,
Vermillion, South Dakota, USA<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Patterns of
evolutionary change in cypriniform fishes: Gill arch development &amp;
evolution</span> </i></b>(poster presentation)</p>

<p class=MsoNormal style='text-align:justify'><b><i><o:p>&nbsp;</o:p></i></b></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><b><span style='font-family:
"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Abstract</span></b><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>:<span
style='mso-spacerun:yes'>  </span></span><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'>Our focus within the Cypriniformes Tree of
Life (CToL) is the comparative morphology and development of the gill arches,
with a long-term interest in understanding the developmental genetic mechanisms
underlying the evolutionary variation in their number, size and shape. We
surveyed skeletal variation in the arches in 65 characters over the 59 species
previously examined by Saitoh et al. (2006).<span style='mso-spacerun:yes'> 
</span>Our morphologically based trees, though poorly resolved, are concordant with
Saitoh et al.’s molecular trees.<span style='mso-spacerun:yes'>  </span>We
documented morphology with digital images that were uploaded to the CToL Portal
and annotated with ontology terms.<span style='mso-spacerun:yes'>  </span>These
were later moved to Morphbank.net, a community image repository, for permanent
storage, retrieval, and reference.<span style='mso-spacerun:yes'> 
</span>Because the images are linked to Teleost Anatomy Ontology (TAO) IDs, the
definition and relationships of each character can be ascertained.<span
style='mso-spacerun:yes'>  </span>The TAO is cross-referenced to the Zebrafish
Information Network database, which contains developmental data for skeletal
elements in zebrafish gill arches, and thus such disparate data
(morphology-genetics) can be traversed.<span style='mso-spacerun:yes'> 
</span>In order to better understand patterns of character evolution, we
studied the developmental morphology of gill arches in the catostomid
cypriniform, <i>Catostomus commersonii</i>.<span style='mso-spacerun:yes'> 
</span>We found qualitatively variable gill arch elements such as
pharyngobranchial 1, pharyngobranchial 4 and the sublingual, are the last to
chondrify.<span style='mso-spacerun:yes'>  </span>The sublingual in <i>C.
commersonii</i> has two cartilaginous precursors that fuse and ossify to form the
single bone in adults.<span style='mso-spacerun:yes'>  </span>This suggests
homology of the sublingual in catostomids to the two sublingual bones of
cobitids and balitorids.<span style='mso-spacerun:yes'>  </span>Intriguing
patterns of fusion and segmentation of the cartilages were discovered including
the individuation of the basihyal and anterior copula through segmentation,
fusion of basibranchials 4 and 5, and fusion of hypobranchial 4 with
ceratobranchial 4.<span style='mso-spacerun:yes'>  </span>Such “fluidity” in
cartilage patterning may be widespread in fishes and requires further
comparative developmental studies.<span style='mso-spacerun:yes'>  </span><o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><b><span style='font-family:
"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Keywords:</span></b><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
</span><span style='font-family:"Times-Roman","serif";mso-bidi-font-family:
Times-Roman'>Cypriniformes, development, evolution, <i>Catostomus commersonii</i>,
morphology</span><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-size:14.0pt;
font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Paula
Mabee<sup>1</sup>*, Jim Balhoff<sup>2</sup>, Wasila Dahdul<sup>1,2</sup>,
Hilmar Lapp<sup>2</sup>, John Lundberg<sup>3</sup>, Peter Midford<sup>4</sup>,
Todd Vision<sup>2,5</sup>, Monte Westerfield<sup>6<o:p></o:p></sup></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><sup><span style='font-size:
14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>1</span></sup><span
style='font-family:"Times-Roman","serif";mso-bidi-font-family:Times-Roman'>Dept.
of Biology, University of South Dakota, Vermillion, South Dakota, USA; </span><sup><span
style='font-size:14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:
TimesNewRomanPSMT'>2</span></sup><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'>National Evolutionary Synthesis Center; </span><sup><span
style='font-size:14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:
TimesNewRomanPSMT'>3</span></sup><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'>: Academy of Natural Sciences, Philadelphia; </span><sup><span
style='font-size:14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:
TimesNewRomanPSMT'>4</span></sup><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'>University of Kansas; </span><sup><span
style='font-size:14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:
TimesNewRomanPSMT'>5</span></sup><span style='font-family:"Times-Roman","serif";
mso-bidi-font-family:Times-Roman'>University of North Carolina; </span><sup><span
style='font-size:14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:
TimesNewRomanPSMT'>6</span></sup><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'>University of Oregon</span><span
style='font-family:"Times-Roman","serif";mso-bidi-font-family:Times-Roman'> Zebrafish
Information Network (zfin.org)</span><span style='font-size:14.0pt;font-family:
"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'><o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-size:14.0pt;
font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><b><i><span style='font-size:
14.0pt;font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>PHENOSCAPE:
Morphology linked to genes using ontologies</span></i></b><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
(oral presentation, </span>Expanding Biodiversity Research Resources in African
Countries)</p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><o:p>&nbsp;</o:p></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><b><span style='font-family:
"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Abstract: </span></b><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Although
systematists have collected comparative morphological data for decades, it has
been rendered in free-form text and is not computable.<span
style='mso-spacerun:yes'>  </span>Even seemingly straightforward comparisons
across taxa, such as examining evolutionary patterns of characters from the
same body region (e.g. head) or of the same quality (e.g. shape changes),
cannot be made broadly without a database and some means of computation.<span
style='mso-spacerun:yes'>  </span>Ontologies, or structured vocabularies with
semantically defined relationships, provide such means and facilitate
integration with other databases, including genetics databases from model
organisms.<span style='mso-spacerun:yes'>  </span>Using ostariophysan fishes,
we are prototyping a system &quot;Phenoscape&quot; with an ontology-based
character database and generalizable tools to integrate evolutionary and model
organism phenotype data.<span style='mso-spacerun:yes'>  </span>This system is
designed to meet the top-priority needs of the evolutionary community
concerning development and evolution of morphology.<span
style='mso-spacerun:yes'>  </span>These include mapping characters on trees and
identification of clades in which traits vary (quick retrieval of character
state distributions), correlation matrices of traits, identification of
candidate genes, and a phenotypic BLAST search to discover taxa with similar
characters.<span style='mso-spacerun:yes'>  </span>Such queries will help
address large-scale questions about the genetic and developmental regulation of
evolutionary morphological transitions. <o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><b><span style='font-family:
"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>Keywords</span></b><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>:
Informatics, zebrafish, cypriniformes, morphology, genetics<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:14.0pt'>A. Manimekalan<sup>*</sup>,
and R.S.C. Jayaraj<o:p></o:p></span></p>

<p class=MsoNormal>Institute of Forest Genetics and Tree Breeding, Coimbatore,
INDIA</p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><b><i><span style='font-size:
14.0pt'>Diversity and distribution of fishes of Periyar Tiger Reserve, Southern
India</span></i></b><span style='font-size:14.0pt'> </span>(poster
presentation)</p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b>Periyar
Tiger Reserve is one of the 27 tiger reserves in India, located in the Western
Ghats in Idukki District of Kerala state in India. The terrain of the Reserve
is undulating and the drainage is dendrite. North-eastern boundary of the
Reserve is a ridge, which also forms the <span
style='mso-spacerun:yes'> </span>boundary between Kerala and Tamil Nadu states
for 90 km. The protected area covers an area of 777 km², out of which a 350 km²
part of the core zone was made into the Periyar National Park and Tiger Reserve.
<span style='mso-spacerun:yes'> </span>The rivers Periyar, Mullaiyar, Pamba and
Azhutha drain through Periyar reserve. River Periyar originates from Chokkampathy
hills and there are several large streams coming from Sundarmala and
Chokkampathy and they the Periyar river. Periyar and Mullaiyar join together at
Mulaikudy. Azhutha river is<b> </b>part of Periyar Tiger reserve and joins with
Pamba river and flows downstream. Pamba river has its catchment mostly in the Ranni
division where there are two dams to impound water for power generation. <span
style='mso-spacerun:yes'> </span>Survey was conducted in 14 sites to know the
distribution and diversity of fishes of Periyar Tiger Reserve<span
style='color:black'>, Kerala</span>. A total of 51 species were collected
belonging to 7 orders, 16 families and 28 genera. Among the 51 species <i>Cyprinidae
i</i>s the dominant family with 38 species (69.9%) compared to other families.
Among the 51 species <span style='color:black'>the fishes mahseer, Tor khudree
is famous as a game fish. The three species <i>Lepidopygopsis typus</i>, <i>Crossocheilis
periyarensis</i> and <i>Puntius micropogan periyarensis</i> are exclusively endemic
to Periyar Tiger Reserve.<span style='mso-spacerun:yes'>  </span></span>The
maximum species richness (S) and diversity (H’) was observed in <span
style='color:black'>Ummikuppan thodu </span>(S-14; H’.95) followed by <span
style='color:black'>Vaikkaparappu thodu </span><span
style='mso-spacerun:yes'> </span>(13; 0.93), <span style='color:black'>Thanikudi
</span><span style='mso-spacerun:yes'> </span>(10; 0.85), Natuthodam estate
(10; 0.85) and Confluence of Puducheri thodu and Pamba river (10; 0.81) river. Low
species richness and diversity was observed in <span style='color:black'>Vaganalla
thodu (3, 0.45).</span><span style='mso-spacerun:yes'>  </span>Four exotic
species (<i><span style='color:black'>Hypophthalmicthys molitrix</span></i><span
style='color:black'>, <i>Ctenopharyngodon idella</i>, <i>Clarius gariepinus</i>,</span><i>
</i>and <i>Oreochromis mossambica </i>(Peters) were recorded in the Periyar
Tiger Reserve. The distributions of all the four exotic species were restricted
to the reservoir at present.<span style='mso-spacerun:yes'>  </span><i><span
style='color:black'>Clarius gariepinus </span></i><span style='color:black'>is
the one of the major catch and has started interbreeding with native clarids. </span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords: </b>Periyar Tiger Reserve, Cyprinids, endemic,
Western Ghats, India</p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;tab-stops:28.0pt 56.0pt 84.0pt 112.0pt 140.0pt 168.0pt 196.0pt 224.0pt 3.5in 280.0pt 308.0pt 336.0pt;
mso-layout-grid-align:none;text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-size:14.0pt;font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'>Donnielle Minor<span style='mso-tab-count:
1'>   </span><o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'>Dillard University, Natural Science
Department, New Orleans, Louisiana USA<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><b><i><span style='font-size:14.0pt;font-family:"TimesNewRomanPS-BoldMT","serif";
mso-bidi-font-family:TimesNewRomanPS-BoldMT'>Phylogenetic Systematics of
Leuciscinae Inferred from Growth Hormone Sequence Data</span></i></b><b><i><span
style='font-family:"TimesNewRomanPS-BoldMT","serif";mso-bidi-font-family:TimesNewRomanPS-BoldMT'>
</span></i></b><span style='font-family:"TimesNewRomanPS-BoldMT","serif";
mso-bidi-font-family:TimesNewRomanPS-BoldMT'>(poster presentation)<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPS-BoldMT","serif";
mso-bidi-font-family:TimesNewRomanPS-BoldMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><b><span style='font-family:"TimesNewRomanPS-BoldMT","serif";
mso-bidi-font-family:TimesNewRomanPS-BoldMT'>Abstract:<span
style='mso-spacerun:yes'>  </span></span></b><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'>The anatomical characters possessed by
the subfamily Leuciscinae of family Cyprinidae make it very unlikely that they
can be recognized as monophyletic. Growth hormone sequence data were researched
to assess if all leuciscines indigenous to North America were monophyletic
separately from those indigenous to Eurasia. <o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'>Using the full length GH DNA sequences,
phylogenetic trees were produced which suggested that North American (NA)
leuciscines are not monophyletic separately from those indigenous to Eurasia
(E). A maximum likelihood tree nested </span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Semotilus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>atromaculatus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
(NA) within a clade with </span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Phoxinus phoxinus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
(E). This tree also nested </span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Notemigonus crysoleucas</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
(NA) within a clade with 5 Eurasian species </span><i><span style='font-family:
"TimesNewRomanPS-ItalicMT","serif";mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Abramis</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>brama</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>,
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Chondrostoma</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>kubanicum</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>,
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Leuciscus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>idus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>,
</span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Rutilus rutilus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>,
and </span><i><span style='font-family:"TimesNewRomanPS-ItalicMT","serif";
mso-bidi-font-family:TimesNewRomanPS-ItalicMT'>Leuciscus idus</span></i><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>.
Based on GH phylogenetic analysis, North American Leuciscinae does not
constitute a monophyletic lineage.<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><b><span style='font-family:"TimesNewRomanPS-BoldMT","serif";
mso-bidi-font-family:TimesNewRomanPS-BoldMT'>Keywords</span></b><span
style='font-family:"TimesNewRomanPSMT","serif";mso-bidi-font-family:TimesNewRomanPSMT'>:
North America, Leuciscinae, Growth Hormone gene.<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:14.0pt;font-family:"TimesNRMT","serif";
mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'>Dorothy Nyingi<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-family:"TimesNRMT","serif";
mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'>National Museums of
Kenya, Niarobi, KENYA<o:p></o:p></span></p>

<p class=MsoNormal><i><span lang=EN-GB style='font-family:"TimesNRMT","serif";
mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal><b><i><span lang=EN-GB style='font-size:14.0pt;font-family:
"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'>Morphometric
and Molecular methods for delineation of taxa and units of conservation</span></i></b><span
lang=EN-GB style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'> (oral presentation, Biodiversity Research Methods)<o:p></o:p></span></p>

<p class=MsoNormal><i><span lang=EN-GB style='font-family:"TimesNRMT","serif";
mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal style='text-align:justify'><b><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'>Abstract: </span></b><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'>Identification of biological units such as species is
the first step to all useful biodiversity studies. This has relied highly on
alfa-taxonomy which provides qualitative and quantitative descriptions,
identification keys and deposition of reference collections in museums.<span
style='mso-spacerun:yes'>  </span>Traditional morphometrics have continued to
be used to acquire numerical comparisons among different levels of taxa.
However, when dealing with ambiguous units of conservation where differentiation
among groups is discrete more powerful discriminative techniques such as
geometric morphometrics and molecular genetics are useful.<span
style='mso-spacerun:yes'>  </span>A combination of both or several methods has
been shown to be very informative in several studies including a systematic study
of 15 populations the Nile tilapia, <i>Oreochromis niloticus</i> from their
natural range of occurrence in Africa using geometric morphometrics and
sequences of mtDNA.<span style='mso-spacerun:yes'>  </span>11 Evolutionary
Significant Units (synonymous with sub-species) were distinguished compared to
7 subspecies described earlier using traditional morphometrics.<span
style='mso-spacerun:yes'>  </span>These inferences have however been
interpreted in reference to important data from meristic counts and linear
morphometrics known to be linked to environmental variability. Nuclear DNA
(microsatellites) studies have also shown evidence on the existence of a
meta-population structure of <i>O. niloticus</i> within the Lake Victoria after
about 50 years since its introduction.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><b><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'>Key words:</span></b><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'> traditional morphometrics, geometric morphometrics,
microsatellites, mtDNA<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:14.0pt'>Ray
Schmidt<sup>1,2</sup>* and Frank Pezold<sup>1<o:p></o:p></sup></span></p>

<p class=MsoNormal style='text-align:justify'><sup><span lang=EN-GB
style='font-family:"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;
mso-ansi-language:EN-GB'>1</span></sup><span lang=EN-GB style='font-family:
"TimesNRMT","serif";mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'>Department
of Biology, Texas A&amp;M University Corpus Christi, Texas; </span><sup><span
lang=EN-GB style='mso-ansi-language:EN-GB'>2</span></sup><span lang=EN-GB
style='mso-ansi-language:EN-GB'>Department of Ecology and Evolutionary Biology,
Tulane University, New Orleans, Louisiana, USA<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><b><i><span lang=EN-GB
style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></i></b></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Morphometric and
molecular variation of mountain catfishes (Amphiliidae, </span></i></b><b><span
style='font-size:14.0pt'>Amphilius<i>) in Guinea, West Africa</i></span><i> </i></b>(poster
presentation)<span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-family:"TimesNRMT","serif";
mso-bidi-font-family:TimesNRMT;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b>Abstract: </b>A morphometric and molecular analysis was
performed on specimens identified as <i>Amphilius</i> from Guinea, West
Africa.<span style='mso-spacerun:yes'>  </span>Specimens and tissue samples of <i>Amphilius
atesuensis</i>, <i>Amphilius rheophilus</i>, and <i>Amphilius platychir</i>
were collected in the streams of the Fouta Djalon, Zone Forestière, and coastal
drainages during expeditions in 2003.<span style='mso-spacerun:yes'>  </span>A
geometric morphometric (GM) analysis showed significant morphometric variation
within the <i>A. platychir</i> and <i>A. rheophilus </i>complex.<span
style='mso-spacerun:yes'>  </span>Results of the standard morphometric analysis
were consistent with GM analysis.<span style='mso-spacerun:yes'>  </span>The
number of gill rakers on the first arch was found to be informative within both
species complexes.<span style='mso-spacerun:yes'>  </span>Phylogenetic analysis
of partial sequences of mitochondrial cytochrome <i>b</i> gene revealed three
distinct clades corresponding to <i>A. platychir, A. atesuensis, </i>and <i>A.
rheophilus</i>.<span style='mso-spacerun:yes'>  </span>Within the <i>A.
platychir</i> clade well-supported deep divergence was observed between
specimens from the Fatala River and those from the Konkouré, Gambie, and
Senegal River basins.<span style='mso-spacerun:yes'>  </span>The <i>A.
rheophilus</i> clade displayed well-supported divergence between specimens from
the Rio Corubal basin and those from the Senegal River basin.<span
style='mso-spacerun:yes'>  </span>Genetic divergence corresponds with the
significant morphometric variation observed with the GM analysis with the <i>A.
rheophilus</i> and <i>A. platychir</i> clades and indicates the presence of
cryptic taxa within both.<span style='mso-spacerun:yes'>  </span>The taxonomic
implications for both species complexes is discussed.<span
style='mso-spacerun:yes'>       </span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: <span lang=FR style='mso-ansi-language:
FR'>Fouta Djalon, <i>Amphilius platychir</i>, <i>Amphilius rheophilus</i>,
cytochrome <i>b<o:p></o:p></i></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span lang=FR style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT;mso-ansi-language:FR'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span lang=FR style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT;mso-ansi-language:FR'><span
style='mso-spacerun:yes'> </span><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Brian L. Sidlauskas*, Richard
L. Mayden, Richard P. Vari</span></p>

<p class=MsoNormal>National Evolutionary Synthesis Center, Durham NC, 27705; Department
of Biology, Saint Louis University, St. Louis, Missouri USA; Richard P. Vari, Division
of Fishes,</p>

<p class=MsoNormal>Smithsonian Institution, Washington, DC, USA</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Building Collaborations
to Reconstruct the Characiform Tree of Life</span> </i></b>(panel presentation,
Building Successful Biodiversity Research Networks)</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b>The
more than 1600 species in the ostariophysan order Characiformes form one of the
most important groups of freshwater fishes in the world and comprise major
portions of the fish diversity of South and Central America and Africa.
Characiform fishes occupy a huge variety of ecological niches ranging from
detritivory to carnivory, support major fisheries, span nearly two orders of
magnitude in body size, and demonstrate repeated convergence with many other
fish groups.<span style='mso-spacerun:yes'>  </span>Despite the ecological,
economic and evolutionary importance of the Characiformes, their systematics
are very poorly known, and even the most comprehensive phylogenetic analyses to
date have included fewer than 10% of the species. To address this knowledge gap
we are assembling a NSF Tree-of-Life proposal to reconstruct the phylogeny of
the entire Characiformes using a combined molecular and morphological approach.
Success of this large enterprise will depend critically upon collaborations
with scientists in many countries, and we anticipate various opportunities for
joint research, collection building, student training and local outreach.<span
style='mso-spacerun:yes'>  </span>We look forward to opening a dialogue about
potential research avenues within the broad scope of this project with the
participants at this Grahamstown meeting. </p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: Characiformes, collaboration,
phylogenetics, systematics, biodiversity</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Brian Sidlauskas<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'>National Evolutionary Synthesis Center, Durham, North
Carolina, USA</p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><b><i><span style='font-size:14.0pt'>Morphometric and
Osteological Methods in Ichthyology</span></i></b> (panel presentation,
Biodiversity Research Methods)</p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b>Anatomical
studies have a long and illustrious history in ichthyology and have played a
key role in reconstructing phylogenetic relationships among species,
recognizing and diagnosing new taxa, and understanding growth, form and
function.<span style='mso-spacerun:yes'>  </span>This presentation will
introduce methods for preparing specimens for morphological study, performing
morphometric analyses using linear measures or geometric coordinates, and
executing phylogenetic analyses using anatomical characters.<span
style='mso-spacerun:yes'>  </span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: Anatomy, Methods, Morphology, Geometric
Morphometrics, Phylogeny</p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPS-BoldMT","serif";
mso-bidi-font-family:TimesNewRomanPS-BoldMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Brian Sidlauskas<o:p></o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'>National Evolutionary Synthesis Center, Durham, North
Carolina, USA</p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><b><i><span style='font-size:14.0pt'>Understanding the
ecomorphological diversification of African distichodontid and citharinid
fishes through phylomorphospace analysis</span> </i></b>(poster presentation)</p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b>The
African characiform fishes in family Distichodontidae vary widely in ecology
and morphology, including species specialized as herbivores, piscivores,
invertivores and even fin-eaters, while species in their sister group
(Citharinidae) are all detritivores with little morphological variation. This
difference in variation between the two groups suggests that some factor has
constrained the ecomorphological evolution of the citharinids or has actively
promoted the diversification of the distichodontids. Projecting a phylogeny
into a multivariate morphospace (a phylomorphospace) reveals that the majority
of skull shape and diet variation within Distichodontidae is contained within a
clade defined by the possession of a novel joint within the lower jaw.<span
style='mso-spacerun:yes'>  </span>This highly diverse clade has a very low
lineage density (a measure of the packing of phylogenetic branches in
morphospace) compared to the citharinids and the remaining distichodontids, all
of which lack the novel joint and have similar skull shapes. Comparison of the
real distichodontid dataset with simulated phylomorphospaces suggests that such
a large difference in lineage densities cannot be reasonably generated by a
null model of Brownian evolution, in which a single rate and mode of
morphological evolution applies across the whole phylogeny. Instead, the
innovative jaw joint possessed by some distichodontids appears to have sparked
a shift in their mode of evolution by releasing anatomical constraints and
allowing the exploration of new ecological niches.<span
style='mso-spacerun:yes'>  </span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: Characiformes, Disparity, Evolution,
Morphometrics, Phylogeny</p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:14.0pt'>Tamatamah, Rashid<o:p></o:p></span></p>

<p class=MsoNormal>Dept. of Fisheries Science and Aquaculture, Univ. of Dar es
Salaam,</p>

<p class=MsoNormal><span lang=PT-BR style='mso-ansi-language:PT-BR'>Dar es
Salaam, TANZANIA<o:p></o:p></span></p>

<p class=MsoNormal><span lang=PT-BR style='mso-ansi-language:PT-BR'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=PT-BR style='mso-ansi-language:PT-BR'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b><i><span style='font-size:14.0pt'>Status of fish
biodiversity research in Tanzania <o:p></o:p></span></i></b></p>

<p class=MsoNormal>(oral presentation, Fish Biodiversity Research Needs and
Challenges in Africa) </p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Abstract:<span style='mso-spacerun:yes'>  </span></b>Fish
biodiversity research in Tanzania dates back to nearly 100 years ago with
earlier works centred on the Great Lakes of Tanganyika, Malawi/Nyassa and
Victoria where a<span style='mso-fareast-font-family:"MS Mincho";mso-fareast-language:
JA'> notably good proportion of species are endemic to those systems. </span>Currently
the bulk of fish biodiversity research in the country is carried out by three
public institutions: Tanzania Fisheries Research Institute (TAFIRI), the
University of Dar es Salaam at its Department of Fisheries Science and
Aquaculture, and the National Museum of Tanzania. <span style='mso-fareast-font-family:
"MS Mincho";mso-fareast-language:JA'>The existing fish biodiversity research is
characterized as being largely donor-funded, widely dispersed, sometimes
inconsistent, and the results that are largely inaccessible to decision makers
and for effective integration within expanding plans for conservation of
freshwater ecosystems and resources. Research institutions have </span>insufficient
laboratory supplies and equipment, insufficient databases and catalogues of
known biodiversity and lack adequate human capacity in some specialized fields
of fish biodiversity research, especially fish taxonomy and molecular genetics.
<span style='mso-fareast-font-family:"MS Mincho";mso-fareast-language:JA'><span
style='mso-spacerun:yes'> </span>Building strong fish biodiversity research
should start with adequate research funding and </span><span style='color:black'>assembling
biodiversity database across national institutions and geographies. In
addition, b</span>ecause many issues related to fish biodiversity transcend
national boundaries, it is often essential for Tanzania to collaborate with
other countries in scientific research and policy development.<span
style='font-size:11.0pt;font-family:"Arial","sans-serif"'><o:p></o:p></span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><b>Keywords</b>: Fish biodiversity research, Tanzania,
research institutions, biodiversity database, research collaboration.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal style='mso-pagination:none;mso-layout-grid-align:none;
text-autospace:none'><span style='font-family:"TimesNewRomanPSMT","serif";
mso-bidi-font-family:TimesNewRomanPSMT'><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
