<html>
<head>
<title>Revision of the Ictiobinae: mtDNA Sequencing</title>
<meta name="generator" content="Frontier 4.2 Mac">
</head><body alink="#008000" bgcolor="#ffffff" link="#000fff" vlink="#ff0000"><table width=600 border=0 cols=2 cellpadding=2 cellspacing=0>	<tr valign=top>		<td width=500 align=left>			<center>			<IMG alt="ictiologo picture" border=0 height=105  src="images/ictiologo.jpg" width =365>			</center>			<p>			<center><h3>Molecular sequencing</h3></center>			<p align=right> 
      <P class=MsoNormal><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt"> 
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; We are&nbsp;using mitochondrial and nuclear genetic markers and modern 
      molecular techniques to assess degrees of DNA sequence variation at the 
      levels of populations, species and genera within subfamily 
      Ictiobinae.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN></SPAN><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">The 
      mitochondrial genome is one of the best studied of all types of DNA 
      (Kocher and Carleton 1997).<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Mitochondrial genes have been used in studies ranging from 
      phylogeny reconstruction and gene evolution to intraspecific 
      phylogeography and gene flow (Avise 1994, 2000, Kocher and Stepien 
      1997).<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Its rapid rate of 
      evolution, clonal inheritance, and lack of recombination have made it a 
      valuable tool for evolutionary studies (Brown 1985, Avise 1994, 
      2000).<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Researchers have used 
      mtDNA to assess degrees of intraspecific genetic diversity and to 
      delineate species boundaries (Frost et al. 1998, Eitner et al. 1999, 
      Tanka-Ueno et al. 1999). <I style="mso-bidi-font-style: normal">Cyt b</I> 
      is perhaps the best studied of all mitochondrial genes, particularly for 
      fishes (Lydeard and Roe 1997).<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>The gene has both conserved and variable regions, and has proven to 
      be useful for investigating relationships of both closely and distantly 
      related species.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Studies of 
      <I style="mso-bidi-font-style: normal">cyt b</I> sequence variation have shown this region 
      to be well adapted for studying evolutionary relationships in 
      Actinopterygian fishes (Lydeard and Roe 1997).<?xml:namespace prefix = o 
      ns = "urn:schemas-microsoft-com:office:office" /><o:p></o:p></SPAN></P>
      <P class=MsoNormal><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt"><SPAN 
      style="mso-tab-count: 1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
      </SPAN>The Growth Hormone gene (GH) is proving to be a useful nuclear 
      genetic marker for fishes and other organisms (Chen et al. 1994).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>The gene comprises five exons 
      (coding regions), and four<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>(non-coding) introns in cypriniformes.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>The exons are evolutionarily 
      conserved, but the introns evolve at a rate suitable for interspecific and 
      intergeneric level studies.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>GH introns have recently been used in subfamily-level phylogenetic 
      studies in salmonid fishes (Oakley and Phillips 1999).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Like salmonids, catostomids are 
      tetraploids (Uyeno and Smith 1972) and thus should have duplicate 
      (paralogous) copies of the GH gene.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Sequencing paralogous copies of GH exons and introns has the 
      potential to reveal interesting facts about ictiobine evolution, as it has 
      for salmonids (Oakley and Philips 1999).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN><o:p></o:p></SPAN></P>
      <P class=MsoBodyTextIndent2><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
       Single-stranded conformation polymorphism (SSCP) offers a simple and 
      inexpensive, yet sensitive (capable of detecting differences in DNA 
      sequence as small as 1-2 bp), method for screening large numbers of 
      individuals, and thus represents an efficient means of assessing 
      population-level variability without sequencing (Sunnucks 2000, Sunnucks 
      et al. 2000).<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>We propose to 
      use SSCP to assess variability in <I>cyt b</I> and GH intron for multiple 
      individuals for each ictiobine study population. As illustrated below, the 
      method is also useful for identifying interspecific hybrids and for 
      determining which individuals need to be cloned and/or sequenced.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN><o:p></o:p></SPAN></P>
      <P class=MsoNormal><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">&nbsp;<o:p></o:p></SPAN></P>
      <P class=MsoNormal><B style="mso-bidi-font-weight: normal"><I 
      style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">Methods<o:p></o:p></SPAN></I></B></P>
      <P class=MsoNormal><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt"><SPAN 
      style="mso-tab-count: 1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
      </SPAN>The following methods were used in the preliminary SSCP, <I 
      style="mso-bidi-font-style: normal">cyt b</I> and GH intron sequencing 
      work described below, and are similar to the methods that will be used for 
      the proposed molecular variation work.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN><o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B 
      style="mso-bidi-font-weight: normal"><I 
      style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Cyt 
      b</SPAN></I></B><B style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      sequencing:</SPAN></B><B style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; mso-bidi-font-family: 'Times New Roman'"> 
      </SPAN></B><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Total 
      genomic DNA was extracted from frozen or ethanol preserved tissues using 
      the DNeasy Tissue Kit<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>(Qiagen, Inc.).<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>The 
      1140 bp <I style="mso-bidi-font-style: normal">Cy -b</I> gene was isolated 
      by PCR.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>We used the 
      oligonucleotides GLU (5'-TAA CCG AGA CCA ATG ACT TG) and THR (5'-- ATC TTC 
      GGA TTA CAA GAC CG) (Brady Porter <I 
      style="mso-bidi-font-style: normal">unpublished</I>) to amplify the 
      gene.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Six additional internal 
      sequencing primers were designed with OLIGO primer analysis software 
      (Molecular Biology Insights, Inc.).<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Reactions were cycled according to the following temperature 
      profile: 94</SPAN><SPAN 
      style="FONT-FAMILY: Symbol; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-ascii-font-family: TIMES; mso-hansi-font-family: TIMES; mso-char-type: symbol; mso-symbol-font-family: Symbol"><SPAN 
      style="mso-char-type: symbol; mso-symbol-font-family: Symbol">�</SPAN></SPAN><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">C 
      for 1 min., 57</SPAN><SPAN 
      style="FONT-FAMILY: Symbol; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-ascii-font-family: TIMES; mso-hansi-font-family: TIMES; mso-char-type: symbol; mso-symbol-font-family: Symbol"><SPAN 
      style="mso-char-type: symbol; mso-symbol-font-family: Symbol">�</SPAN></SPAN><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">C 
      for 1 min., and 72</SPAN><SPAN 
      style="FONT-FAMILY: Symbol; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-ascii-font-family: TIMES; mso-hansi-font-family: TIMES; mso-char-type: symbol; mso-symbol-font-family: Symbol"><SPAN 
      style="mso-char-type: symbol; mso-symbol-font-family: Symbol">�</SPAN></SPAN><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">C 
      for 1:15 min., for 32 cycles.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>PCR products were isolated with the QIAquick PCR Purification 
      Kit<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>(Qiagen, Inc) and used in 
      cycle sequencing reactions (Applied Biosystems Inc.) according to the 
      manufacturer's recommendations.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Excess dye terminators, primers, and nucleotides were removed by 
      gel filtration (Edge Biosystems) prior to sequencing. Sequences in both 
      directions were determined with an ABI 373A Automated Sequencer.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Reactions were electrophoresed on 
      6% polyacrylamide gels in 7 M urea (Sooner Scientific).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Raw sequence chromatograms of 
      approximately 400 bp length were assembled into contigs and edited to 
      resolve ambiguities using Sequencher 4.1 (Gene Codes, Inc.). Sequences 
      were aligned to <I style="mso-bidi-font-style: normal">Myxocyprinus 
      asiaticus</I>, an Asian Cycleptine (GenBank AF036176). Amino acid 
      sequences were determined and analyzed with MacVector 4.0 (Oxford 
      Molecular Ltd.).<o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B 
      style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">SSCP:</SPAN></B><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      </SPAN><SPAN style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">A 300 bp 
      fragment of GH Intron B fragment was isolated by PCR using oligonucleotide 
      primers specific to copy I of the gene.<SPAN 
      style="mso-spacerun: yes">&nbsp;&nbsp; </SPAN>The 50 �l PCR reactions 
      contained 100 ng total genomic DNA, 1.5mM MgCL2, 1X platinum taq reaction 
      buffer (Invitrogen), 2mM each dNTP, 40 pmol each primer, and 1U platinum 
      taq DNA polymerase (Invitrogen). The PCR thermal profile consisted of an 
      initial 94�C hot start for 2 min., then 30 cycles of 94�C for 30 sec, 56�C 
      for 30 sec, and 72�C for 30 sec.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Five �l of the PCR reaction was electrophoresed in a 1% agarose gel 
      for 1 hr at 100V. The gel was stained with ethidium bromide for 10 min and 
      products were visualized with a UV transilluminator to check the quality 
      of the PCR reactions.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>3-5 ul 
      of the reaction was diluted with 5 �l of SSCP loading buffer (95% 
      formamide, 20mM EDTA, 0.05% bromophenol blue, 0.05% xylene cyanol). The 
      mixture was then heated at 95�C for 3 min and immediately chilled on ice 
      for 5 min. The samples were loaded onto a 19cm (W) X 16 cm (L), 10% 
      polyacrylamide gel (37.5:1 acrylamide: bisacrylamide) and electrophoresed 
      at 5W for 15 hrs. The gel apparatus was kept cool during the run by 
      placing it in a 4�C refrigerator.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>At the end of the run, the glass plates were disassembled and the 
      gel was silver stained.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>SSCP 
      gels were dried at room temperature overnight, then visualized with a 
      light box and photographed with a digital camera.</SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B 
      style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">GH 
      sequencing: </SPAN></B><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">Fresh pituitaries were 
      harvested from adult <I style="mso-bidi-font-style: normal">I. bubalus</I> 
      and immediately frozen at -70 C until use. Total RNA was extracted from 
      100 mg of pituitary tissue using a modified acid guanidinium 
      thiocyanate-phenol-chloroform protocol (Ultraspec, Biotecx) and the amount 
      and purity of RNA was determined by spectrophotometry. First strand cDNA 
      was reverse-transcribed from total RNA using the Super script RT-PCR kit 
      (Invitrogen) and 2 �l of the reaction served as template for subsequent 
      PCR amplifications. Approximately 550bp of the coding sequence of the GH 
      gene was amplified using Oligo 6.6 designed (MBI) degenerate primers from 
      alignments of cypriniform GH sequences deposited on GenBank. PCR reactions 
      contained 2 �l of first strand reaction, 1.5mM MgCl<SUB>2</SUB>, 1X PCR 
      buffer, 0.2mM each dNTP, 20 pmols of primers GHF and GHR (sequences 
      available on request), and 0.5 U platinum <I 
      style="mso-bidi-font-style: normal">taq</I> DNA polymerase (Invitrogen). 
      The thermal profile consisted of an initial "hot start" of 94 C for 2 min 
      to activate enzyme. Then 30 cycles of 94 C for 30 sec, 52 C for 30 sec, 
      and 72 C for 1 min. PCR products were electrophoresed in 1% agarose at 
      100V for 1hour and stained with ethidium bromide. PCR products were 
      visualized with a 312nm variable intensity transilluminator.<SPAN 
      style="mso-spacerun: yes">&nbsp;&nbsp; </SPAN><o:p></o:p></SPAN></P>
      <P class=MsoNormal><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt"><SPAN 
      style="mso-tab-count: 1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
      </SPAN>PCR products were inserted and ligated to pCR4-TOPO vectors, and 
      transformed into TOP 10 competent cells (TOPO TA Cloning Kit, Invitrogen). 
      Plasmid DNA was extracted from bacterial colonies with Qiagen QIaprep kit 
      and insert size was checked by agarose gel electrophoresis after digestion 
      with <I style="mso-bidi-font-style: normal">EcoR1</I>.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>We sequenced 20 clones to 
      discriminate PCR or Taq errors. Multiple clones were sequenced using dye 
      terminator cycle sequencing reactions (Applied Biosystems) and sequences 
      from both directions were determined with an ABI 373A Automated Sequencer. 
      Additionally, several internal sense and antisense sequencing primers were 
      designed and used in sequencing reactions to assure complete coverage of 
      clones with high quality sequence data. Raw sequence chromatograms of 400 
      bp length were assembled into contigs and edited using Sequencher 4.1 
      (Gene Codes). Intron boundaries were determined by aligning cDNA sequences 
      to other cypriniform GH sequences. Intron spanning primers were designed 
      to include approximately 50bp of flanking exon so these areas could be 
      aligned with known GH sequences to confirm that the correct intron 
      sequence was isolated. <o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">To amplify and compare 
      GH intron sequences across several ictiobine taxa, PCR reactions were 
      performed with 100ng total genomic DNA using copy specific primers that 
      anneal to exons 2 and 3 of the representative Ictiobine GH gene. PCR 
      protocol was similar to that described earlier except annealing 
      temperature was set to 55 C. Small aliquots of the PCR reactions were run 
      on agarose gels to check the quality and size of PCR products. PCR 
      products were purified using Qiagen QIAquick PCR extraction kits. PCR 
      products were directly sequenced as described above. 
<o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B 
      style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Phylogenetic 
      analysis:</SPAN></B><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      Pairwise sequence divergence and transition-transversion ratios were 
      calculated to examine patterns of variation in <I 
      style="mso-bidi-font-style: normal">cyt b </I>and GH. Phylogenetic trees 
      were generated with PAUP*4.0 (Swofford 1998) under Maximum Parsimony (MP) 
      and Maximum Likelihood (ML) models. Robustness of inferred nodes from 
      parsimony analysis was assessed with bootstrap analysis.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>The best fit model of nucleotide 
      substitution for ML analysis was determined through likelihood ratio tests 
      using the program MODELTEST 3.04 (Posada and Crandall 1998).<SPAN 
      style="mso-spacerun: yes">&nbsp;&nbsp; </SPAN>Posterior probabilities 
      (i.e. confidence) of nodes in ML trees were estimated using MRBAYES 
      (Huelsenbeck and Rondquiust, in press)</SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"></SPAN>&nbsp;</P>
      <P class=MsoNormal style="mso-outline-level: 1"><B 
      style="mso-bidi-font-weight: normal"><I 
      style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-FAMILY: TIMES; mso-bidi-font-family: 'Times New Roman'">Preliminary 
      Results</SPAN></I></B><B style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; mso-bidi-font-family: 'Times New Roman'">&nbsp;<o:p></o:p></SPAN></B></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B 
      style="mso-bidi-font-weight: normal"><I 
      style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Cyt 
      b</SPAN></I></B><B style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      sequencing:</SPAN></B><B style="mso-bidi-font-weight: normal"><SPAN 
      style="FONT-FAMILY: TIMES; mso-bidi-font-family: 'Times New Roman'"> 
      </SPAN></B><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">To 
      date, complete <I style="mso-bidi-font-style: normal">cyt b</I> sequence 
      data have been obtained for 20 individuals representing all of the 
      currently recognized extant species in subfamily Ictiobinae (and multiple 
      populations for most of the species) and five catostomid outgroup 
      taxa.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Sequences for an 
      additional 5 outgroups (an Asian sucker, 2 minnows and 2 cobitids) were 
      obtained from Genbank. These data were used to infer relationships within 
      subfamily Ictiobinae (<A 
      href="http://www.tubri.org/museum/ictiobin/ictiobinephylogenetics.pdf">Bart
      et al. <I style="mso-bidi-font-style: normal">in review</I></A>).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Space limits will only allow us to 
      give general results of this study.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Reviewers who wish to view the complete manuscript may visit 
      http://www.tubri.org/museum/ictiobin/MPE2002.htm/. <o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">At 
      the level of ictiobines, <I>cyt b</I> sequences contain 172 variable 
      sites, the vast majority of which (94%) are at the 3rd codon position. The 
      Ti/Tv ratio is 7:1. Within genus <I 
      style="mso-bidi-font-style: normal">Ictiobus</I> 74 sites are variable 
      (89% 3<SUP>rd</SUP> position substitutions) and transitions outnumber 
      transversions 20:1. There are 28 variable sites within genus <I 
      style="mso-bidi-font-style: normal">Carpiodes</I><SPAN 
      style="mso-bidi-font-style: italic">, all but one of which is </SPAN>at 
      the 3rd position, and the Ti/Tv ratio is 47:1. Sequences show evidence of 
      saturation only for 3<SUP>rd</SUP> position transitions and the distinct 
      anti-G bias at the 2<SUP>nd</SUP> and 3<SUP>rd</SUP> codon positions 
      typical of mitochondrial genes.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>None of the base substitutions in genus <I 
      style="mso-bidi-font-style: normal">Carpiodes</I> alter the amino acid 
      sequence of the <I style="mso-bidi-font-style: normal">cyt b</I> 
      protein.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN><I 
      style="mso-bidi-font-style: normal">Ictiobus</I> sequences show four amino 
      acid differences (2 conservative, 2 nonconservative).</SPAN><SPAN 
      style="FONT-FAMILY: TIMES; mso-bidi-font-family: 'Times New Roman'"><o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Sequence 
      divergences among <I style="mso-bidi-font-style: normal">Carpiodes</I> 
      species are low, ranging from 0-1.3% (mean = 0.85%). <I 
      style="mso-bidi-font-style: normal">Cyt b</I> sequences for the two <I 
      style="mso-bidi-font-style: normal">C. velifer</I> specimens are 
      identical. Sequences for three Miss. River Basin <I 
      style="mso-bidi-font-style: normal">C. cyprinus</I> specimens and two of 
      four Miss River Basin <I style="mso-bidi-font-style: normal">C. carpio</I> 
      specimens differ at only two positions, suggesting that the two <I 
      style="mso-bidi-font-style: normal">C. carpio</I> specimens are hybrids 
      with <I style="mso-bidi-font-style: normal">C. cyprinus cyt b </I><SPAN 
      style="mso-bidi-font-style: italic">haplotypes</SPAN>. Sequences for two 
      other Miss. R. <I style="mso-bidi-font-style: normal">C. carpio</I> 
      specimens are 0.96 to 1.16% divergent from the typical Miss R. <I 
      style="mso-bidi-font-style: normal">C. cyprinus</I> haplotype, 0.43 to 
      0.61% divergent from the Miss. R. <I 
      style="mso-bidi-font-style: normal">C. velifer</I> haplotype, and 0.35% 
      divergent from each other.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>We 
      regard these latter <I style="mso-bidi-font-style: normal">C. carpio 
      </I>sequences as more typical of the true <I 
      style="mso-bidi-font-style: normal">C. carpio</I> haplotype. 
      <o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Sequence 
      divergence among <I style="mso-bidi-font-style: normal">Ictiobus</I> 
      species average 1.51%.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>However, much of this is due to the high divergences between <I 
      style="mso-bidi-font-style: normal">I. labiosus </I>and other <I 
      style="mso-bidi-font-style: normal">Ictiobus</I> species (average of 
      6%).<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Divergences among <I 
      style="mso-bidi-font-style: normal">I. bubalus</I>, <I 
      style="mso-bidi-font-style: normal">I. cyprinellus</I> and <I 
      style="mso-bidi-font-style: normal">I. niger</I> average 0.46%.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Divergence within these latter 
      three species - even among widely separated populations of <I 
      style="mso-bidi-font-style: normal">I. bubalus</I> (inclusive of <I 
      style="mso-bidi-font-style: normal">I. meridionalis</I>)<I 
      style="mso-bidi-font-style: normal"> </I>- are much lower.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN><I 
      style="mso-bidi-font-style: normal">Cyt b</I> sequences for <I 
      style="mso-bidi-font-style: normal">Ictiobus bubalus</I> specimens from 
      the R�o San Fernando in northeastern Mexico and the Amite River (Lake 
      Pontchartrain Basin), just east of the lower Mississippi River, are 
      identical. Sequences for the <I style="mso-bidi-font-style: normal">I. 
      meridionalis</I> specimen from R�o Usumacinta (extreme southeastern 
      Mexico) and <I style="mso-bidi-font-style: normal">I. bubalus</I> from the 
      Upper Mississippi River differ by only one base.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Mean sequence divergence within <I 
      style="mso-bidi-font-style: normal">I. bubalus</I> (inclusive of<I 
      style="mso-bidi-font-style: normal"> I. meridionalis</I>) is 0.22%.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Two base substitutions separate 
      the sequences of Ohio and Upper Mississippi R. specimens of <I 
      style="mso-bidi-font-style: normal">I. niger</I>, and only one base 
      substitution separates the sequences of Ohio and Upper Mississippi R. 
      specimens of <I style="mso-bidi-font-style: normal">I. 
      cyprinellus</I>.<SPAN style="mso-spacerun: yes">&nbsp;&nbsp; </SPAN>The 
      <I>c</I><I style="mso-bidi-font-style: normal">yt b</I> sequence for the 
      Amite River specimen of <I style="mso-bidi-font-style: normal">I. 
      cyprinellus </I><SPAN style="mso-bidi-font-style: italic">i</SPAN>s 
      unusual in that it is 0.57% divergent on average from other <I 
      style="mso-bidi-font-style: normal">I. cyprinellus</I> sequences, but only 
      0.09% and 0.25% divergent, respectively, from upper Mississippi and Ohio 
      R. <I style="mso-bidi-font-style: normal">I. niger</I> <I>c</I><I 
      style="mso-bidi-font-style: normal">yt b </I>sequences, suggesting that 
      the Amite River <I style="mso-bidi-font-style: normal">I. cyprinellus</I> 
      specimen is expressing an <I style="mso-bidi-font-style: normal">I. 
      niger</I> <I>c</I><I style="mso-bidi-font-style: normal">yt b</I> DNA 
      haplotype.<o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Sequence 
      divergences among <I style="mso-bidi-font-style: normal">Carpiodes</I> and 
      <I style="mso-bidi-font-style: normal">Ictiobus</I> species average 10.5% 
      overall.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Sequence divergences 
      between ictiobines and the two Cycleptines (</SPAN><I 
      style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">Myxocyprinus 
      asiaticus</SPAN></I><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      and </SPAN><I style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">Cycleptus 
      elongatus</SPAN></I><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">)</SPAN><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      average 14.8%. As a group, Ictiobines are as divergent from Moxostomini as 
      from minnows (17.2-17.4%).<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Ictiobines are most divergent from loaches among non-catostomid 
      outgroups (20-20.4%).<o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-ALIGN: justify; TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Of 
      the 478 variable sites at the level of all taxa, 416 are parsimony 
      informative.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>At the level of 
      ictiobines, 131 of the172 variable sites are parsimony informative.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>However, phylogenetic signal falls 
      off considerably within ictiobine genera. Within genus <I 
      style="mso-bidi-font-style: normal">Carpiodes</I>, 14 of the 28 variable 
      sites are parsimony informative. In genus <I 
      style="mso-bidi-font-style: normal">Ictiobus</I>, only 8 of 74 variable 
      sites are parsimony informative.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>The likelihood ratio test in Modeltest identified GTR+I+G as the 
      model of base substitution that best fit the data.</SPAN><SPAN 
      style="FONT-FAMILY: TIMES; mso-bidi-font-family: 'Times New Roman'"><SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN></SPAN><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">ML 
      analysis and MP analysis excluding 3<SUP>rd</SUP> position transitions 
      (which show evidence of saturation) produced trees with nearly identical 
      topologies.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Only the ML tree 
      is shown (<A 
      href="http://www.tubri.org/museum/ictiobin/figures/figure5.jpg"
      target=_blank>Fig 5</A>).<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Both trees resolve Family Catostomidae, Subfamily Ictiobinae, and 
      genera <I style="mso-bidi-font-style: normal">Carpiodes</I> and <I 
      style="mso-bidi-font-style: normal">Ictiobus</I> as monophyletic with high 
      bootstrap support.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>However, 
      the trees fail to resolve a number of the species complexes as 
      monophyletic (namely <I style="mso-bidi-font-style: normal">C. carpio</I>, 
      <I style="mso-bidi-font-style: normal">C. cyprinus</I>, <I 
      style="mso-bidi-font-style: normal">I. cyprinellus</I>, and <I 
      style="mso-bidi-font-style: normal">I. niger</I>).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>We attribute the low sequence 
      divergence and low phylogenetic resolution observed at the species level 
      for <I>cyt b</I> to intrageneric, interspecific hybridization, 
      particularly within the Mississippi River Basin.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN><o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-ALIGN: justify; TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">The 
      hypothesis that hybridization is influencing patterns of <I>c</I><I 
      style="mso-bidi-font-style: normal">yt b</I> sequence variation in 
      ictiobines is entirely plausible.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Hybrids among <I style="mso-bidi-font-style: normal">I. 
      bubalus</I>, <I style="mso-bidi-font-style: normal">I. cyprinellus</I>, 
      and <I style="mso-bidi-font-style: normal">I. niger </I>have been observed 
      in nature (Robison and Buchanan 1989, Etnier and Starnes 1993) and 
      produced in experimental ponds (Stevenson 1964).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Moreover, all three of the species 
      are known to hybridize introgressively in reservoirs (Johnson and Minckley 
      1969).<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>Four morphologically 
      divergent catostomine sucker species in the Klamath River Basin (presently 
      assigned to three genera) similarly show very low levels of mitochondrial 
      DNA sequence divergence, which has also been attributed to interspecific 
      hybridization (see reports under "Genetics" at <A 
      href="http://www.mp.usbr.gov/kbao/esa/">http://www.mp.usbr.gov/kbao/esa/</A>).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">SSCP:</SPAN></B><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'"> 
      <A href="http://www.tubri.org/museum/ictiobin/figures/Figure6.jpg"
      target=_blank>Figure 6 </A>shows preliminary SSCP results for 5 
      individuals each of <I>I. bubalus, I. cyprinellus </I>and <I>I. niger</I> 
      from the Wisconsin River (Upper Mississippi River Basin).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Three different conformational 
      patterns are evident in <I>I. bubalus</I> and <I>I. cyprinellus</I>, and 
      four patterns are evident in <I>I. niger</I>. <I><SPAN 
      style="mso-spacerun: yes">&nbsp;</SPAN></I>Similarities are evident 
      when<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>patterns are 
      compared<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>across species 
      (e.g., lanes 2, 7, and 13), suggesting that some of individuals are 
      interspecific hybrids. Nevertheless, species-specific SSCP patterns can be 
      identified for specimens of <I>I. bubalus</I> (lanes 3 and 5), <I>I. 
      cyprinellus</I> (lanes 6,9 and 10), and <I>I. niger</I> (12 and 14).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>By sequencing GH I Intron B for 
      specimens with these patterns, together with specimens from other 
      populations of these species, <I>I. cyprinellus</I> (3 populations) formed 
      a monophyletic group related to <I>I. niger</I>, with these species sister 
      in turn to <I>I. bubalus</I>. Thus, SSCP enabled us to assess genetic 
      variability within populations, identify interspecific hybrids, and 
      identify individuals for sequencing that allowed us to better resolve 
      species of <I>Ictiobus</I> as monophyletic. We expect similar results 
      using SSCP for <I>cyt b</I>.</SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><B><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">GH 
      sequencing:</SPAN></B><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt"> Sequencing of clones 
      revealed two unique cDNAs for <I>I. bubalus</I>, which differed in 
      nucleotide sequence by 5.3%.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>The validity of this result was suggested two ways.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>First, all amino acid changes were 
      conservative and none caused termination of the GH protein.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Second, the paralogs differed at 
      codon 123 by a single nucleotide, agreeing with previously identified 
      paralogs of <I style="mso-bidi-font-style: normal">Carassius auratus</I> 
      GH genes which have a divergent number of total cysteine residues (more 
      specifically either 4 or 5 based on a C-&gt;T transition at nucleotide 
      position 369, Law et al. 1996). Moreover, amplification of genomic DNA 
      using primers spanning GH Intron C (i.e., primers binding to Exons 3 and 
      4) produced two discrete product bands (<A 
      href="http://www.tubri.org/museum/ictiobin/figures/figure7.jpg"
      target=blank>Fig. 7</A>), indicating that the two paralogous copies of GH 
      intron C are highly divergent.<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>The two amplified products have been cloned and sequenced, and have 
      characteristics (intron splice donor and acceptor sites) indicating that 
      neither is a psuedogene.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>GH I 
      and II are to 93% identical in the short exonic region amplified.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>The sequences then diverge rapidly 
      starting at 10 bp past the splicing sites; the variation involves 6 of the 
      first 30 bp, and 16 of the next 30 bp (data not shown).<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>We were able to use this 
      nucleotide variation in the coding regions to design copy-specific 
      primers.</SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">Intron B, measuring 
      300 bp in length, was amplified and bi-directionally sequenced for seven 
      <I style="mso-bidi-font-style: normal">Ictiobus</I> specimens (<I 
      style="mso-bidi-font-style: normal">I. bubalus</I> from upper Miss. R., 
      Lake Pontchartrain and R�o San Fernando, <I 
      style="mso-bidi-font-style: normal">I. cyprinellus</I> from upper Miss. 
      R., Ohio R. and Lake Pontchartrain, and <I 
      style="mso-bidi-font-style: normal">I. niger </I>from upper Miss. 
      R.).<SPAN style="mso-spacerun: yes">&nbsp;&nbsp; </SPAN>Comparisons of the 
      sequences reveals more interspecific divergence (average of 3%) than 
      observed for <I style="mso-bidi-font-style: normal">cyt b</I> sequences of 
      the same taxa (average 0.4%).<SPAN style="mso-spacerun: yes">&nbsp; 
      </SPAN>Interpopulational divergences average 0.4%.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>We also detected taxon-specific 
      indels, which may also prove useful for resolving species and population 
      level questions. Thus, we are confident that sequences from GH introns 
      will give us resolution at the lowest taxonomic levels (species complexes 
      to populations) that we currently lack with <I 
      style="mso-bidi-font-style: normal">cyt b</I>.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Moreover, having sequences from 
      nuclear and cytoplasmic genes will enable us to measure associations 
      between these genomes and test for cytonuclear disequilibria.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>Cytonuclear disequilibria provide 
      an array of conceptual tools useful for examining hybrid zones, including 
      levels of gene flow, age of reproductive barriers, directionality of 
      crosses between hybridizing species, levels of assortative mating, and 
      mechanisms of selection on hybrids (Arnold 1993).</SPAN></P>
      <P class=MsoNormal><B style="mso-bidi-font-weight: normal"><I 
      style="mso-bidi-font-style: normal"><SPAN 
      style="FONT-FAMILY: TIMES; FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: 'Times New Roman'">Proposed 
      Molecular Work </SPAN></I></B></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">We request support to 
      isolate total DNA from 1,800 specimens of ictiobines.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>In sampling populations of each of 
      the currently recognized species, we will remove fin clips from up to 20 
      individuals. Half of the specimens will be preserved for detailed 
      morphological study, the rest will be photographed and released. We have 
      identified areas of the distribution of each of the currently recognized 
      species that we will initially target for assessing genetic variability 
      (60 areas in all, <A 
      href="http://www.tubri.org/museum/ictiobin/figures/table1.jpg"
      target="">Table 1</A>). The areas were selected to represent what we 
      believe to be the major zones of differentiation of ictiobine species 
      complexes based our morphological studies and evidence from other groups 
      of fishes (Wiley and Mayden 1985, chapters in Hocutt and Wiley 1986, 
      Eisenhour 1999). Wide-ranging species are represented by more areas. Areas 
      to be sampled along the Gulf and Atlantic coasts encompass a number of 
      rivers that lack freshwater connections today.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>We are targeting at least one area 
      of the distribution of each of the species where other congeners do not 
      occur to get a sense of genetic divergence without the potential for 
      hybridization. <o:p></o:p></SPAN></P>
      <P class=MsoNormal style="TEXT-INDENT: 0.5in"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt">We will run SSCP 
      analysis on <I style="mso-bidi-font-style: normal">cyt b</I> and GH 
      introns for 10 individuals from 3 populations from each of these 
      areas.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>SSCP will give us a 
      general sense of genetic diversity within populations and will allow us to 
      identify the variants we need to sequence.<SPAN 
      style="mso-spacerun: yes">&nbsp; </SPAN>The combination of <I 
      style="mso-bidi-font-style: normal">cyt b</I> and GH intron sequences, and 
      cytonuclear disequilibrium analysis will give us the resolution we need to 
      assess evolutionary relationships among haplotypes and impacts of 
      interspecific hybridization. <B 
      style="mso-bidi-font-weight: normal"><o:p></o:p></B></SPAN></P>
      <P class=MsoNormal 
      style="mso-pagination: widow-orphan; tab-stops: 0in .5in 1.0in 1.5in 2.0in 2.5in 3.0in 3.5in 4.0in 4.5in 5.0in 5.5in 6.0in"><SPAN 
      style="FONT-SIZE: 11pt; mso-bidi-font-size: 10.0pt"><SPAN 
      style="mso-tab-count: 1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
      </SPAN>The goal of the molecular portion of this study is to produce 
      independent phylogenies at the levels of Subfamily Ictiobinae, genera <I 
      style="mso-bidi-font-style: normal">Carpiodes</I> and <I 
      style="mso-bidi-font-style: normal">Ictiobus</I>, and currently recognized 
      species complexes.<SPAN style="mso-spacerun: yes">&nbsp; </SPAN>We will 
      select outgroups for phylogenetic analyses based on existing phlyogenetic 
      hypotheses of basal catostomid relationships (Smith 1992, Harris and 
      Mayden 2001, <A 
      href="http://www.tubri.org/museum/ictiobin/ictiobinephylogenetics.pdf">Bart
      et al. <I style="mso-bidi-font-style: normal">in review</I></A>). 
      <o:p></o:p></SPAN></P>			<p align=right>			<A href="allozymes.html">Prev</A> | <A href="litcited.html">Next</A></p>		</td>		<td width=100 align=right bgcolor="#99cc99">			<center>			<IMG alt="tree picture" border=0 height=77  src="images/tree.gif" width =44>			</center>			<font face=arial,geneva>			<p>			<A href="default.html">Summary</A>			</p>			<p>			<A href="intro.html">Introduction</A>			</p>			<p>			<A href="carpiodes.html">Carpiodes</A>			</p>			<p>			<A href="ictiobus.html">Ictiobus</A>			</p>			<p>			<A href="distribution.html">Distribution</A>			</p>			<p>			<A href="field.html">Field Work</A>			<p>			<A href="extmorph.html">External Morphology</A>			</p>			<p>			<A href="typeloc.html">Types &amp; Their Location</A>			</p>			<p>			<A href="morphometrics.html">Morphometrics</A>			</p>			<p>			<A href="allozymes.html">Allozymes &amp; Isozymes</A>			</p>
      <P>			<A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>			<p>			<A href="litcited.html">Literature Cited</A>			</p>			<p>			<A href="pubpres.html">Publications &amp; Presentations</A>			</p>			<p>			<a href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's Research</a>			</p>			<p>			<a href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum of Natural History</a>			</p>
      <P>&nbsp;</P>
      <P><FONT face=arial,geneva> </P>
      <P><A href="default.html">Summary</A> </P>
      <P><A href="intro.html">Introduction</A> </P>
      <P><A href="carpiodes.html">Carpiodes</A> </P>
      <P><A href="ictiobus.html">Ictiobus</A> </P>
      <P><A href="distribution.html">Distribution</A> </P>
      <P><A href="field.html">Field Work</A> 
      <P><A href="extmorph.html">External Morphology</A> </P>
      <P><A href="typeloc.html">Types &amp; Their Location</A> </P>
      <P><A href="morphometrics.html">Morphometrics</A> </P>
      <P><A href="allozymes.html">Allozymes &amp; Isozymes</A> </P>
      <P><A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>
      <P><A href="litcited.html">Literature Cited</A> </P>
      <P><A href="pubpres.html">Publications &amp; Presentations</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's
      Research</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum
      of Natural History</A> </P>
      <P>			</font>&nbsp;</P>
      <P><FONT face=arial,geneva> </P>
      <P><A href="default.html">Summary</A> </P>
      <P><A href="intro.html">Introduction</A> </P>
      <P><A href="carpiodes.html">Carpiodes</A> </P>
      <P><A href="ictiobus.html">Ictiobus</A> </P>
      <P><A href="distribution.html">Distribution</A> </P>
      <P><A href="field.html">Field Work</A> 
      <P><A href="extmorph.html">External Morphology</A> </P>
      <P><A href="typeloc.html">Types &amp; Their Location</A> </P>
      <P><A href="morphometrics.html">Morphometrics</A> </P>
      <P><A href="allozymes.html">Allozymes &amp; Isozymes</A> </P>
      <P><A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>
      <P><A href="litcited.html">Literature Cited</A> </P>
      <P><A href="pubpres.html">Publications &amp; Presentations</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's
      Research</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum
      of Natural History</A> </P>
      <P></FONT><FONT face=arial,geneva>&nbsp;</P>
      <P><A href="default.html">Summary</A> </P>
      <P><A href="intro.html">Introduction</A> </P>
      <P><A href="carpiodes.html">Carpiodes</A> </P>
      <P><A href="ictiobus.html">Ictiobus</A> </P>
      <P><A href="distribution.html">Distribution</A> </P>
      <P><A href="field.html">Field Work</A> 
      <P><A href="extmorph.html">External Morphology</A> </P>
      <P><A href="typeloc.html">Types &amp; Their Location</A> </P>
      <P><A href="morphometrics.html">Morphometrics</A> </P>
      <P><A href="allozymes.html">Allozymes &amp; Isozymes</A> </P>
      <P><A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>
      <P><A href="litcited.html">Literature Cited</A> </P>
      <P><A href="pubpres.html">Publications &amp; Presentations</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's
      Research</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum
      of Natural History</A> </P>
      <P></FONT><FONT face=arial,geneva>&nbsp;</P>
      <P><A href="default.html">Summary</A> </P>
      <P><A href="intro.html">Introduction</A> </P>
      <P><A href="carpiodes.html">Carpiodes</A> </P>
      <P><A href="ictiobus.html">Ictiobus</A> </P>
      <P><A href="distribution.html">Distribution</A> </P>
      <P><A href="field.html">Field Work</A> 
      <P><A href="extmorph.html">External Morphology</A> </P>
      <P><A href="typeloc.html">Types &amp; Their Location</A> </P>
      <P><A href="morphometrics.html">Morphometrics</A> </P>
      <P><A href="allozymes.html">Allozymes &amp; Isozymes</A> </P>
      <P><A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>
      <P><A href="litcited.html">Literature Cited</A> </P>
      <P><A href="pubpres.html">Publications &amp; Presentations</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's
      Research</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum
      of Natural History</A> </P>
      <P></FONT><FONT face=arial,geneva>&nbsp;</P>
      <P><A href="default.html">Summary</A> </P>
      <P><A href="intro.html">Introduction</A> </P>
      <P><A href="carpiodes.html">Carpiodes</A> </P>
      <P><A href="ictiobus.html">Ictiobus</A> </P>
      <P><A href="distribution.html">Distribution</A> </P>
      <P><A href="field.html">Field Work</A> 
      <P><A href="extmorph.html">External Morphology</A> </P>
      <P><A href="typeloc.html">Types &amp; Their Location</A> </P>
      <P><A href="morphometrics.html">Morphometrics</A> </P>
      <P><A href="allozymes.html">Allozymes &amp; Isozymes</A> </P>
      <P><A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>
      <P><A href="litcited.html">Literature Cited</A> </P>
      <P><A href="pubpres.html">Publications &amp; Presentations</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's
      Research</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum
      of Natural History</A> </P>
      <P></FONT><FONT face=arial,geneva>&nbsp;</P>
      <P><A href="default.html">Summary</A> </P>
      <P><A href="intro.html">Introduction</A> </P>
      <P><A href="carpiodes.html">Carpiodes</A> </P>
      <P><A href="ictiobus.html">Ictiobus</A> </P>
      <P><A href="distribution.html">Distribution</A> </P>
      <P><A href="field.html">Field Work</A> 
      <P><A href="extmorph.html">External Morphology</A> </P>
      <P><A href="typeloc.html">Types &amp; Their Location</A> </P>
      <P><A href="morphometrics.html">Morphometrics</A> </P>
      <P><A href="allozymes.html">Allozymes &amp; Isozymes</A> </P>
      <P><A href="http://www.tubri.org/museum/ictiobin/DNA.html" >Molecular Sequencing</A> </P>
      <P><A href="litcited.html">Literature Cited</A> </P>
      <P><A href="pubpres.html">Publications &amp; Presentations</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/reshank.html">Hank Bart's
      Research</A> </P>
      <P><A href="http://www.tubri.org/museum/museum/museum.html">Tulane Museum
      of Natural History</A> </P>
      <P></FONT>&nbsp;</P></FONT>		</td>	</tr></table><p><IMG src="counter.count$tulane/ictiobin"></p>
</body>
</html>
<!--
#fatPage
#version 1
#docs http://www.scripting.com/fatPages/faq.html
#adrObject user.websites.ictiobin.mtDNA
#netAddress 129.81.102.17
#objectCreated 1/9/98; 2:55:35 PM
#objectModified 1/22/99; 3:37:03 PM
#lastBuildTime 2/25/99; 4:29:46 PM
-->

