<html><head><title>Revision of the Ictiobinae: Need, Material, Procedures</title><meta name="generator" content="Frontier 4.2 Mac"></head><body alink="#008000" bgcolor="#FFFFFF" link="#000FFF" vlink="#FF0000"><table width=600 border=0 cols=2 cellpadding=2 cellspacing=0>	<tr valign=top>		<td width=500 align=left>			<center>			<img src="images/ictiologo.jpg" height=105 width=365 border=0  alt="ictiologo picture" border=0>			</center>			<p>			<p><b>Need, material, and procedures for modern systematic revison</b></p> <p>Although the nomenclature of extant Ictiobinae is now relatively stable, modern workers attempting to describe diversity within state and region fish guides continue to cite problems with the current taxonomy (Burr and Warren 1986, Robison and Buchanan 1988, Jenkins and Burkhead 1994, Etnier and Starnes 1994, Mettee et al. 1996). The problems stem from the fact that: 1) most of the species are poorly diagnosed; 2) only one comprehensive treatment of geographic variation has been attempted for these wide ranging species (<i>C. carpio,</i> Hubbs and Black 1940) and even this study was conducted without the benefit of sufficient material; and 3) it is difficult to reliably assign young and juveniles to species or even genus. </p><p><a href="tables/table1.html">Table 1</a> is a summary of material presently in the Tulane Fish Collection, arranged by species and drainage, and showing the total number of specimens from each drainage. This analysis also identifies instances where more specimens are needed. The listings of drainages for each species closely approximate the distributions of each of the species (compare to distributioin maps in Lee et al. 1980). In most instances, coastal drainages and major tributaries of the Mississippi River are represented by adequate numbers of specimens. We have also taken inventory of material in other museums and feel that sufficient material is now available to address all of the problems alluded to above. </p> <p>Tulane holdings are weakest for northern tributaries of the Mississippi River, Canada (Lake Manitoba and the Laurentian Great Lakes), the north Atlantic Coast and Mexico. Ample northern and midwestern material is available at the Museum of Zoology, Field Museum of Natural History, National Museum of Natural History, Academy of Natural Sciences, and the Illinois Natural History Survey. Ample southwestern material is available at Texas Memorial Collection and the Museum of Southwestern Biology. The Field Museum and Museum of Zoology have considerable material of the Mexican forms, but more material is needed, especially from lowland portions of some of the rivers draining into the Gulf of Mexico (Robert R. Miller, pers. comm.).</p><p>For the past 10 years, we have been accumulating material from some of the sparsely represented areas of the upper midwest, and the Gulf and Atlantic coasts. We have been assisted in this effort by a number of other ichthyologists and fishery biologists (<a href="table2.html">Table 2</a>). Beginning this spring, we will contact ichthyologists in Texas and New Mexico (Dean Hendrickson, Kirk Winemiller, Doyle Moser, Steve Plantania) for assistance collecting specimens of all forms from the Brazos to the Rio Grande. With the help of Dr. Salvador Constrera-Balderas, Universidad Autonoma de Nuevo Leon, we are establishing collaborations with Mexican ichthologists (Dr. Eduardo Diaz Pardo, IPN and Dr. Hector Espinosa, UNAM) that will enable us to collect fresh material of the forms in Mexico. </p><p>Tulane Fish Collection also has a fair amount of skeletal material available for study, and we are in the process of preparing additional skeletons and accumulating liver and muscle tissues for molecular sequencing work. </p><p>We will use the preserved material to examine geographic variation (meristic and morphometric), and to establish reliable characters for the identification of species and different life stages. Our approach to morphological aspects of the revision will be to first work through material in the Tulane Fish Collection species by species, and then travel to other museums to study types (<a href="tables/table3.html">Table 3</a>) and work up material Tulane is lacking. Ictiobines are large fish that are not conveniently shipped. Travelling to other collections will give us the opportunity to collect fresh material for osteology and tissues for molecular sequencing work.</p><p>Suttkus has been studying skeletons and gathering data on a standard assortment of meristic and morphometric characters from Tulane specimens for a number of years. Bart has been accumulating tissues and learning molecular sequencing techniques. Bart will be on a one semester sabbatical leave starting in January 1998 and intends to devote all of this time and much of the following summer to assisting Suttkus with the study of Tulane material, collecting additional tissues and specimens, and doing molecular sequencing work in the laboratory of Dr. David Hurley (see supporting letter).</p><p>Bart is requesting support for release from teaching responsibilities in Fall 1998 (one-month academic year salary). Starting in late summer 1998, the PI's will travel to museums to study types and other specimens. Locations of types are given in <a href="tables/table3.html">Table 3</a>. Having already worked through material at Tulane, we should have a good sense of the characters useful for distinguishing the different species. This will make our work at other institutions more efficent, it will aid us in working with types (some of which are old and in poor condition), and it will reduce the likelihood that we will have to return to the collection to re-examine the material. </p><p><i>Morphometrics</i></p><p>Ictiobines can be differentiated on the basis of morphometric characters, such as proportions of the head and body, size and shape of the eye, length of the snout and position of the nares, size and height of the fins, and size, and position of the mouth (<a href="figures.html">Fig. 1</a>). We have developed a sense of the morphometric variation within species of <i>Carpiodes</i> and to a lesser extent <i>Ictiobus</i> from collecting new material and considering patterns of vicariance in other similarly distributed fish taxa. It appears that the range of <i>C. velifer</i> comprises as many as five distinct forms: a mostly northern form in upper Mississippi River and Ohio River Basin that is deep bodied and possess extremely long anterior dorsal fin rays; a form mostly in western tributaries of the Mississippi River that is not as deep bodied and has a shorter dorsal fin. A form in coastal drainages east of the Mississippi River from Lake Pontchartrain to the Mobile Basin. An form in Gulf drainages east of Mobile Bay, form on the middle to lower Atlantic slope. <i>C. cyprinus</i> comprises as many as many as four distinct forms: two in the upper Mississippi River Basin including an uncommon, elongate form with a short dorsal fin (<i>C. forbesi</i> Hubbs); this form is sympatric in the upper Mississippi Basin with a more northern and northeasterly distributed form that may extend to the St. Lawrence River and upper Atlantic Slope (this form is deep bodied and possess a long dorsal fin); a form in the eastern Gulf; and form on the lower Altantic slope. The range of <i>C. carpio</i> encompasses an elongate western form (<i>C. carpio elongatus</i>) which apparently occupies lower western tributaries of the Mississippi River and integrades with the larger, deep bodied, small headed <i>C.c. carpio</i> of central and eastern tributaries of the Mississippi River (Hubbs and Black 1940). </p><p>We are less certain about diversity within species of <i>Ictiobus.</i> The range of <i>I. bubalus</i> appears to encompass two forms (Mississippi River and Eastern Gulf), however little is know about western Gulf and Mexican populations (the southern most of which was described by Gunther in 1868 as a distinct species). We can recognize only one form of <i>I. cyprinellus</i> based on body shape, but have noted considerable variabily across the range. <i>I. niger</i> is enigmatic to many ichthyologists and fishery biologists, some of whom have suggested that it is a hyrid between <i>I bubalus</i> and <i>I. cyprinellus.</i> It is almost exactly intermediate between the two in body form. However, that fact that its range extends westward into Mexico without one of its parent species (<i>I. cyprinellus</i>), leaves little doubt of its validity. We have not studied any of the western species, but understand from colleagues that the forms in the Rio Grande and possibly the isolated form in the Rio Nazas are distinctive. Smith's (1992) asertion that <i>I. labiosus</i> is sister to <i>I. cyprinellus</i> is difficult to rationalize with their widely disjunct distributions. </p><p>We will use modern morphometric methods to quantify differences in body morphometry and attempt to interpret it taxonomically. A video camera will be used to capture images of types and samples from throughout the ranges of all of the recognized forms. The images will be uploaded into a PC equipped with a frame grabber board and running MorphoSys software (Meacham and Duncan, 1990) for registry of 24 landmarks selected to approximate important axes along which shape of ictiobines varies (<a href="figures.html">Fig. 2</a>). A digital camera will be used to capture images of types and specimens examined at other museums. The camera images will be downloaded to a notebook computer to preview image quality and ensure images are adequate for landmark registry. We will preview image quality. Partial warp analysis and superposition analysis will be used to transform the coordinate variables into shape variables. Principal components analysis and canonical variates analysis will then be used to summarize shape variation within and among species of Ictiobines. Assuming the species are diagnosable by other characters, this analysis should resolve questions shape variability within species. By integrating information on environmental conditions specimens are drawn (lake vs. stream, turbid vs. clear stream, etc.) it should resolve questions about the environmental basis (if any) for some of this variation.</p><p><b><i>Molecular sequencing</i></b></p><p>We will use modern methods of molecular systematics to establish an independent data set on intra- and interspecific genetic variation within the subfamily Ictiobinae. A pilot study is under way to demonstrate the usefulness of the mitochondrial cytochrome b gene for inferring sequence evolution varition at different levels of universality within the subfamily. The following text describes procedures we are using in the pilot study and the point this work has advanced to at this writing. This is done to give the reader as close as a sense as possible of the methods and portion of the mitochondrial genome that will be used in this study. The pilot work is continuing and results are posted on the <a href="mtDNA.html">mtDNA</a> page as they become available. </p> <p><i>DNA Extraction</i></p><p>We removed liver tissue from upper Mississippi Basin (Wisconsin River) and Lake Pontchartrain Drainage specimens of <i>Carpiodes velifer,</i> <i>C. cyprinus</i> (including tissue from a upper Mississippi form believed to be <i>C. forbesi</i>) <i>Ictiobus bubalus,</i> <i>I. cyprinellus,</i> and <i>I. niger</i> (upper Miss. only). Mitochondrial DNA was extracted using the G Nome DNA Isolation Kit (Bio101, Inc.). Microgram quantities of liver tissue were homogenized in 1.85 ml of Cell Suspension Solution. Following homogenization, RNase Mixx (50&micro;l) and Cell Lysis/Denaturing Solution (100&micro;l) were added to the homogenate. The mixture was then incubated at 55&deg;C for 15 minutes. Protease Mixx was added to the solution and incubated at 55&deg;C for 30 to 120 minutes. Following incubation, 500&micro;l of Salt-Out Mixture was added and the mixture was refrigerated at 4&deg;C for 10 minutes. Nuclear debris and DNA were microcentrifuged at 13,000 xg for 10 minutes and the supernatant retained. Two milliliters of TE buffer and 8 mls of 100% ethanol were added to each tube. The entire mixture was centrifuged for 15 minutes at 1000-1500xg, air dryed, suspended in 500 &micro;l of water and stored at 4&deg;C.</p><p><i>Primers and region of the genome to be sequenced</i></p><p>Primers designed to amplify a 1200 segment of the cytochrome-b gene region of mitochondrial DNA were obtained from Tom Near of the Illinois Natural History Survey (<a href="table4.html">Table 4</a>). These highly conserved primers have been used to amplify homologous regions of the mitochondrial genome of Actinopterygian fishes as diverse and minnows and darters (T. Near, pers. comm.). </p><p>The cytochrome b gene has proven to be a phylogenetically useful region of the mitochondrial genome in a variety of vertebrate groups including mammals (Irwin et al. 1991, Krajewski et al. 1992), birds (Krajewski and Fetzner 1994), reptiles (Hedges et al. 1991, Lamb et al. 1994) and fishes (See review by Lydeard and Roe 1997). Because this gene has both conservative and variable regions, it has proven to be a useful for investigating relationships of both closely and distantly related taxa. The cytochrome b gene will be used to generate a phylogenetic tree for the ictobine genera <i>Ictiobus</i> and <i>Carpiodes.</i> Studies of cytochrome b sequence variation have shown that this region is well adapted for studying the evolution and phylogenetic relationships of Actinopterygian groups (Song 1994, Schmidt and Gold 1995; Lydeard et al. 1995; Wiley and Hagen 1997, Lydeard and Roe 1997). Phylogenetic methods will be used to assess the intra- and interspecific variation among the taxa of the subfamily Ictiobinae</p><p><i>Polymerase Chain Reaction (PCR) and sequencing</i></p><p>Double stranded product was produced using 100 ng of total DNA, 40 pmol of each primer, 50 mM MgCl<sub>2</sub>, 5&micro;l 10x reaction buffer (Perkin Elmer), 1 &micro;l dNTP's, 1 &micro;l <i>Taq</i> polymerase, and water to bring the reactions to a final volume of 50&micro;l. Reactions were cycled according to the following temperature profile: 94&deg;C for 1 min., 45&deg;C for 1 min., and 72&deg;C for 2 min. (ca. 35 cycles). A portion (10&micro;l) of the double stranded amplification product was electrophoresed in a 0.7% agarose gel and visualized under UV light following ethidium bromide staining (Fig. 3). The band containing the amplified DNA was cut from the agarose gel and purified with the Gene Clean II Kit (Bio101, Inc.).</p><p>DNA samples were precipitated in ethanol. Two microliters of 3M NaOAc (pH 5.4) and 55&micro;l of 100% of cold ethanol were added to each 1.5 ml tube, mixed, and stored on ice for 15 minutes. Each sample was microfuge at 13 krpm for 20 minutes. Liquid was pipetted from off each sample and 400ml of 70% ethanol was then added. Again, the liquid was removed from each sample. Tubes were dryed in a Speedvac for 10 minutes. </p><p>The next phase of work has yet to be performed, but the propcedures to be used are described for completeness. Three microliters of formamide will be added to each tube. Fragments will be electrophoretically seperated on 6% polyacrylamide gels (8 M urea). Samples will be electrophoresed on the Applied Systems 373A DNA Sequencing System. </p><p><i>Data analysis</i></p><p>Sequence Navigator (Applied Biosystems, Inc.) will be used to create mulitple simultaneous aligments of nucleotide sequences and an electropherogram. Sequence data will be converted into a series of informative, unordered character states (Fitch 1971) and used to generate a phylogenetic tree. Variance in sequence length due to insertion or deletion will be coded as a fifth character state.</p><p>Cladistic analyses will be performed on the aligned sequence data. Both parsimony and maximum likelihood analyses will be utilized. The software programs PAUP 3.1 (Swofford 1993) and the DNAML module in PHYLIP (Felsenstein 1989) will be employed in the analysis. A check of the ratio of transitions to tranversion will guide decisions of relative weighting (normally 10:1 in favor of transversions). Phylogenetic trees will be rooted by the outgroup method (Maddison et al. 1984) using sequences from <i>Cyprinus carpio</i> and <i>Cycleptus elongatus.</i></p>			</p>			<p align=right>			Prev | Next		</td>		<td width=100 align=right bgcolor="#99CC99">			<center>			<img src="images/tree.gif" height=77 width=44 border=0  alt="tree picture" border=0>			</center>			<font face=arial,geneva>			<p>			<a href="default.html">Summary</a>			</p>			<p>			<a href="intro.html">Introduction</a>			</p>			<p>			<a href="carpiodes.html">Carpiodes</a>			</p>			<p>			<a href="ictiobus.html">Ictiobus</a>			</p>			<p>			<a href="distribution.html">Distribution</a>			</p>			<p>			<a href="fieldwork.html">Field Work</a>			<p>			<a href="extmorph.html">External Morphology</a>			</p>			<p>			<a href="morphometrics.html">Morphometrics</a>			</p>			<p>			<a href="allozymes.html">Allozymes & Isozymes</a>			</p>			<p>			<a href="mtDNA.html">mtDNA Sequencing</a>			</p>			<p>			<a href="litcited.html">Literature Cited</a>			</p>			<p>			<a href="pubpres.html">Publications & Presentations</a>			</p>			</font>		</td>	</tr></table><p><img src="counter.count$tulane/ictiobin"></p></body></html><!--#fatPage#version 1#docs http://www.scripting.com/fatPages/faq.html#adrObject user.websites.ictiobin.need#netAddress 129.81.102.17#objectCreated 1/8/98; 3:04:12 PM#objectModified 2/5/98; 2:06:35 PM#lastBuildTime 1/15/99; 5:08:50 PM-->